diff -Naurp buildroot-2017.05-rc2/package/alsa-utils/alsa-utils.mk buildroot-2017.05-rc2-patch/package/alsa-utils/alsa-utils.mk
--- buildroot-2017.05-rc2/package/alsa-utils/alsa-utils.mk	2017-05-17 10:27:16.027844496 +0200
+++ buildroot-2017.05-rc2-patch/package/alsa-utils/alsa-utils.mk	2017-09-12 11:18:38.000000000 +0200
@@ -48,40 +48,4 @@ else
 ALSA_UTILS_CONF_OPTS += --disable-bat
 endif
 
-ALSA_UTILS_TARGETS_$(BR2_PACKAGE_ALSA_UTILS_ALSACONF) += usr/sbin/alsaconf
-ALSA_UTILS_TARGETS_$(BR2_PACKAGE_ALSA_UTILS_ALSACTL) += usr/sbin/alsactl
-ALSA_UTILS_TARGETS_$(BR2_PACKAGE_ALSA_UTILS_ALSALOOP) += usr/bin/alsaloop
-ALSA_UTILS_TARGETS_$(BR2_PACKAGE_ALSA_UTILS_ALSAMIXER) += usr/bin/alsamixer
-ALSA_UTILS_TARGETS_$(BR2_PACKAGE_ALSA_UTILS_ALSATPLG) += usr/bin/alsatplg
-ALSA_UTILS_TARGETS_$(BR2_PACKAGE_ALSA_UTILS_AMIDI) += usr/bin/amidi
-ALSA_UTILS_TARGETS_$(BR2_PACKAGE_ALSA_UTILS_AMIXER) += usr/bin/amixer
-ALSA_UTILS_TARGETS_$(BR2_PACKAGE_ALSA_UTILS_APLAY) += usr/bin/aplay usr/bin/arecord
-ALSA_UTILS_TARGETS_$(BR2_PACKAGE_ALSA_UTILS_BAT) += usr/bin/alsabat
-ALSA_UTILS_TARGETS_$(BR2_PACKAGE_ALSA_UTILS_IECSET) += usr/bin/iecset
-ALSA_UTILS_TARGETS_$(BR2_PACKAGE_ALSA_UTILS_ACONNECT) += usr/bin/aconnect
-ALSA_UTILS_TARGETS_$(BR2_PACKAGE_ALSA_UTILS_ALSAUCM) += usr/bin/alsaucm
-ALSA_UTILS_TARGETS_$(BR2_PACKAGE_ALSA_UTILS_APLAYMIDI) += usr/bin/aplaymidi
-ALSA_UTILS_TARGETS_$(BR2_PACKAGE_ALSA_UTILS_ARECORDMIDI) += usr/bin/arecordmidi
-ALSA_UTILS_TARGETS_$(BR2_PACKAGE_ALSA_UTILS_ASEQDUMP) += usr/bin/aseqdump
-ALSA_UTILS_TARGETS_$(BR2_PACKAGE_ALSA_UTILS_ASEQNET) += usr/bin/aseqnet
-ALSA_UTILS_TARGETS_$(BR2_PACKAGE_ALSA_UTILS_SPEAKER_TEST) += usr/bin/speaker-test
-
-define ALSA_UTILS_INSTALL_TARGET_CMDS
-	mkdir -p $(TARGET_DIR)/var/lib/alsa
-	for i in $(ALSA_UTILS_TARGETS_y); do \
-		$(INSTALL) -D -m 755 $(STAGING_DIR)/$$i $(TARGET_DIR)/$$i || exit 1; \
-	done
-	if [ -x "$(TARGET_DIR)/usr/bin/speaker-test" ]; then \
-		mkdir -p $(TARGET_DIR)/usr/share/alsa/speaker-test; \
-		mkdir -p $(TARGET_DIR)/usr/share/sounds/alsa; \
-		cp -rdpf $(STAGING_DIR)/usr/share/alsa/speaker-test/* $(TARGET_DIR)/usr/share/alsa/speaker-test/; \
-		cp -rdpf $(STAGING_DIR)/usr/share/sounds/alsa/* $(TARGET_DIR)/usr/share/sounds/alsa/; \
-	fi
-	if [ -x "$(TARGET_DIR)/usr/sbin/alsactl" ]; then \
-		mkdir -p $(TARGET_DIR)/usr/share/; \
-		rm -rf $(TARGET_DIR)/usr/share/alsa/; \
-		cp -rdpf $(STAGING_DIR)/usr/share/alsa/ $(TARGET_DIR)/usr/share/alsa/; \
-	fi
-endef
-
 $(eval $(autotools-package))
diff -Naurp buildroot-2017.05-rc2/package/alsa-utils/Config.in buildroot-2017.05-rc2-patch/package/alsa-utils/Config.in
--- buildroot-2017.05-rc2/package/alsa-utils/Config.in	2017-05-17 10:27:16.027844496 +0200
+++ buildroot-2017.05-rc2-patch/package/alsa-utils/Config.in	2017-09-12 11:20:32.000000000 +0200
@@ -9,6 +9,7 @@ menuconfig BR2_PACKAGE_ALSA_UTILS
 	depends on !BR2_STATIC_LIBS # uses dlfcn.h
 	select BR2_PACKAGE_ALSA_LIB
 	select BR2_PACKAGE_GETTEXT if BR2_NEEDS_GETTEXT_IF_LOCALE
+	select BR2_PACKAGE_PKGCONF
 	help
 	  This package contains the command line utilities for the ALSA
 	  project.
diff -Naurp buildroot-2017.05-rc2/package/autoconf/autoconf.mk buildroot-2017.05-rc2-patch/package/autoconf/autoconf.mk
--- buildroot-2017.05-rc2/package/autoconf/autoconf.mk	2017-05-17 10:27:16.027844496 +0200
+++ buildroot-2017.05-rc2-patch/package/autoconf/autoconf.mk	2017-05-28 18:40:12.000000000 +0200
@@ -17,7 +17,9 @@ HOST_AUTOCONF_CONF_ENV = \
 	ac_cv_prog_gnu_m4_gnu=no
 
 HOST_AUTOCONF_DEPENDENCIES = host-m4 host-libtool
+AUTOCONF_DEPENDENCIES = m4 libtool
 
+$(eval $(autotools-package))
 $(eval $(host-autotools-package))
 
 # variables used by other packages
diff -Naurp buildroot-2017.05-rc2/package/autoconf/Config.in buildroot-2017.05-rc2-patch/package/autoconf/Config.in
--- buildroot-2017.05-rc2/package/autoconf/Config.in	1970-01-01 01:00:00.000000000 +0100
+++ buildroot-2017.05-rc2-patch/package/autoconf/Config.in	2017-05-28 18:51:24.000000000 +0200
@@ -0,0 +1,8 @@
+config BR2_PACKAGE_AUTOCONF
+	bool "autoconf"
+	select BR2_PACKAGE_M4
+	select BR2_PACKAGE_LIBTOOL
+	help
+	  autoconf
+
+	  http://....
diff -Naurp buildroot-2017.05-rc2/package/automake/automake.mk buildroot-2017.05-rc2-patch/package/automake/automake.mk
--- buildroot-2017.05-rc2/package/automake/automake.mk	2017-05-17 10:27:16.027844496 +0200
+++ buildroot-2017.05-rc2-patch/package/automake/automake.mk	2017-09-15 15:45:31.000000000 +0200
@@ -11,6 +11,7 @@ AUTOMAKE_LICENSE = GPL-2.0+
 AUTOMAKE_LICENSE_FILES = COPYING
 
 HOST_AUTOMAKE_DEPENDENCIES = host-autoconf
+AUTOMAKE_DEPENDENCIES = autoconf
 
 ACLOCAL_HOST_DIR = $(HOST_DIR)/usr/share/aclocal
 
@@ -27,6 +28,7 @@ endef
 HOST_AUTOMAKE_POST_INSTALL_HOOKS += GTK_DOC_M4_INSTALL
 HOST_AUTOMAKE_POST_INSTALL_HOOKS += HOST_AUTOMAKE_MAKE_ACLOCAL
 
+$(eval $(autotools-package))
 $(eval $(host-autotools-package))
 
 # variables used by other packages
diff -Naurp buildroot-2017.05-rc2/package/automake/Config.in buildroot-2017.05-rc2-patch/package/automake/Config.in
--- buildroot-2017.05-rc2/package/automake/Config.in	1970-01-01 01:00:00.000000000 +0100
+++ buildroot-2017.05-rc2-patch/package/automake/Config.in	2017-05-28 18:06:57.000000000 +0200
@@ -0,0 +1,7 @@
+config BR2_PACKAGE_AUTOMAKE
+	bool "automake"
+	select BR2_PACKAGE_AUTOCONF
+	help
+	  automake
+
+	  http://....
diff -Naurp buildroot-2017.05-rc2/package/bash/bash.mk buildroot-2017.05-rc2-patch/package/bash/bash.mk
--- buildroot-2017.05-rc2/package/bash/bash.mk	2017-05-17 10:27:16.027844496 +0200
+++ buildroot-2017.05-rc2-patch/package/bash/bash.mk	2017-09-16 17:55:19.000000000 +0200
@@ -22,6 +22,8 @@ BASH_CONF_ENV += \
 	bash_cv_func_sigsetjmp=present \
 	bash_cv_printf_a_format=yes
 
+BASH_CONF_OPTS += --without-bash-malloc
+
 # The static build needs some trickery
 ifeq ($(BR2_STATIC_LIBS),y)
 BASH_CONF_OPTS += --enable-static-link --without-bash-malloc
diff -Naurp buildroot-2017.05-rc2/package/binutils/binutils.mk buildroot-2017.05-rc2-patch/package/binutils/binutils.mk
--- buildroot-2017.05-rc2/package/binutils/binutils.mk	2017-05-17 10:27:16.027844496 +0200
+++ buildroot-2017.05-rc2-patch/package/binutils/binutils.mk	2017-09-15 16:03:02.000000000 +0200
@@ -44,8 +44,6 @@ BINUTILS_DISABLE_GDB_CONF_OPTS = \
 BINUTILS_CONF_OPTS = \
 	--disable-multilib \
 	--disable-werror \
-	--host=$(GNU_TARGET_NAME) \
-	--target=$(GNU_TARGET_NAME) \
 	--enable-install-libiberty \
 	--enable-build-warnings=no \
 	$(BINUTILS_DISABLE_GDB_CONF_OPTS) \
@@ -105,21 +103,13 @@ define BINUTILS_BUILD_CMDS
 	$(TARGET_MAKE_ENV) $(TARGET_CONFIGURE_ARGS) $(MAKE) $(BINUTILS_MAKE_OPTS) -C $(@D)
 endef
 
-# We just want libbfd, libiberty and libopcodes,
-# not the full-blown binutils in staging
 define BINUTILS_INSTALL_STAGING_CMDS
-	$(TARGET_MAKE_ENV) $(MAKE) -C $(@D)/bfd DESTDIR=$(STAGING_DIR) install
-	$(TARGET_MAKE_ENV) $(MAKE) -C $(@D)/opcodes DESTDIR=$(STAGING_DIR) install
-	$(TARGET_MAKE_ENV) $(MAKE) -C $(@D)/libiberty DESTDIR=$(STAGING_DIR) install
+	$(TARGET_MAKE_ENV) $(MAKE) -C $(@D) DESTDIR=$(STAGING_DIR) install
 endef
 
-# If we don't want full binutils on target
-ifneq ($(BR2_PACKAGE_BINUTILS_TARGET),y)
 define BINUTILS_INSTALL_TARGET_CMDS
-	$(TARGET_MAKE_ENV) $(MAKE) -C $(@D)/bfd DESTDIR=$(TARGET_DIR) install
-	$(TARGET_MAKE_ENV) $(MAKE) -C $(@D)/libiberty DESTDIR=$(STAGING_DIR) install
+	$(TARGET_MAKE_ENV) $(MAKE) -C $(@D) DESTDIR=$(TARGET_DIR) install
 endef
-endif
 
 ifneq ($(ARCH_XTENSA_CORE_NAME),)
 define BINUTILS_XTENSA_OVERLAY_EXTRACT
diff -Naurp buildroot-2017.05-rc2/package/bison/bison.mk buildroot-2017.05-rc2-patch/package/bison/bison.mk
--- buildroot-2017.05-rc2/package/bison/bison.mk	2017-05-17 10:27:16.027844496 +0200
+++ buildroot-2017.05-rc2-patch/package/bison/bison.mk	2017-05-28 18:38:42.000000000 +0200
@@ -10,5 +10,7 @@ BISON_SITE = $(BR2_GNU_MIRROR)/bison
 BISON_LICENSE = GPL-3.0+
 BISON_LICENSE_FILES = COPYING
 HOST_BISON_DEPENDENCIES = host-m4
+BISON_DEPENDENCIES = m4
 
+$(eval $(autotools-package))
 $(eval $(host-autotools-package))
diff -Naurp buildroot-2017.05-rc2/package/bison/Config.in buildroot-2017.05-rc2-patch/package/bison/Config.in
--- buildroot-2017.05-rc2/package/bison/Config.in	1970-01-01 01:00:00.000000000 +0100
+++ buildroot-2017.05-rc2-patch/package/bison/Config.in	2017-05-28 18:37:35.000000000 +0200
@@ -0,0 +1,10 @@
+config BR2_PACKAGE_BISON
+	bool "bison"
+	depends on BR2_TOOLCHAIN_HAS_THREADS
+	help
+	  bison
+
+	  https://......
+
+comment "iw needs a toolchain w/ threads"
+	depends on !BR2_TOOLCHAIN_HAS_THREADS
diff -Naurp buildroot-2017.05-rc2/package/cifs-utils/cifs-utils.mk buildroot-2017.05-rc2-patch/package/cifs-utils/cifs-utils.mk
--- buildroot-2017.05-rc2/package/cifs-utils/cifs-utils.mk	2017-05-17 10:27:16.027844496 +0200
+++ buildroot-2017.05-rc2-patch/package/cifs-utils/cifs-utils.mk	2017-06-02 20:38:17.000000000 +0200
@@ -10,6 +10,8 @@ CIFS_UTILS_SITE = http://ftp.samba.org/p
 CIFS_UTILS_LICENSE = GPL-3.0+
 CIFS_UTILS_LICENSE_FILES = COPYING
 
+CIFS_UTILS_CONF_OPTS += --with-pamdir=/lib/security
+
 ifeq ($(BR2_TOOLCHAIN_SUPPORTS_PIE),)
 CIFS_UTILS_CONF_OPTS += --disable-pie
 endif
diff -Naurp buildroot-2017.05-rc2/package/cmake/Config.in buildroot-2017.05-rc2-patch/package/cmake/Config.in
--- buildroot-2017.05-rc2/package/cmake/Config.in	2017-05-17 10:27:16.027844496 +0200
+++ buildroot-2017.05-rc2-patch/package/cmake/Config.in	2017-05-27 15:00:41.000000000 +0200
@@ -8,7 +8,27 @@ config BR2_PACKAGE_CMAKE_ARCH_SUPPORTS
 		BR2_i386      || BR2_x86_64      || BR2_xtensa
 
 config BR2_PACKAGE_CMAKE
-	bool
+	bool "cmake"
+	depends on BR2_PACKAGE_CMAKE_ARCH_SUPPORTS
+	depends on BR2_TOOLCHAIN_GCC_AT_LEAST_4_7 # from jsoncpp
+	depends on BR2_USE_WCHAR # libarchive
+	depends on BR2_INSTALL_LIBSTDCPP
+	depends on !BR2_STATIC_LIBS
+	depends on BR2_TOOLCHAIN_HAS_THREADS_NPTL # libuv
+	depends on BR2_USE_MMU # libuv
+	depends on BR2_TOOLCHAIN_HAS_SYNC_4 # libuv
+	select BR2_PACKAGE_ZLIB
+	select BR2_PACKAGE_JSONCPP
+	select BR2_PACKAGE_LIBCURL
+	select BR2_PACKAGE_LIBARCHIVE
+	select BR2_PACKAGE_EXPAT
+	select BR2_PACKAGE_BZIP2
+	select BR2_PACKAGE_XZ
+	select BR2_PACKAGE_LIBUV
+	help
+	  CMake
+
+	  http://www.cmake.org/
 
 config BR2_PACKAGE_CMAKE_CTEST
 	bool "ctest"
diff -Naurp buildroot-2017.05-rc2/package/coreutils/coreutils-8.27-uname-1.patch buildroot-2017.05-rc2-patch/package/coreutils/coreutils-8.27-uname-1.patch
--- buildroot-2017.05-rc2/package/coreutils/coreutils-8.27-uname-1.patch	1970-01-01 01:00:00.000000000 +0100
+++ buildroot-2017.05-rc2-patch/package/coreutils/coreutils-8.27-uname-1.patch	2017-09-15 16:08:41.000000000 +0200
@@ -0,0 +1,170 @@
+Submitted by:            DJ Lucas (dj_at_linuxfromscratch_dot_org)
+Date:                    2012-04-21
+Initial Package Version: 8.16
+Upstream Status:         Rejected
+Origin:                  Based on Gentoo patch
+Description:             Makes uname -m output more descriptive
+
+diff -Naurp coreutils-8.16-orig/src/uname.c coreutils-8.16/src/uname.c
+--- coreutils-8.16-orig/src/uname.c	2012-04-22 20:02:39.000000000 +0000
++++ coreutils-8.16/src/uname.c	2012-04-22 20:02:50.000000000 +0000
+@@ -49,6 +49,11 @@
+ # include <mach-o/arch.h>
+ #endif
+ 
++#if defined(__linux__)
++# define USE_PROCINFO
++# define UNAME_HARDWARE_PLATFORM
++#endif
++
+ #include "system.h"
+ #include "error.h"
+ #include "quote.h"
+@@ -153,6 +158,117 @@ Print machine architecture.\n\
+   exit (status);
+ }
+ 
++#if defined(USE_PROCINFO)
++
++# if defined(__s390__) || defined(__s390x__)
++#  define CPUINFO_FILE    "/proc/sysinfo"
++#  define CPUINFO_FORMAT  "%64[^\t :]%*[ :]%256[^\n]%c"
++# else
++#  define CPUINFO_FILE    "/proc/cpuinfo"
++#  define CPUINFO_FORMAT  "%64[^\t:]\t:%256[^\n]%c"
++# endif
++
++# define PROCINFO_PROCESSOR      0
++# define PROCINFO_HARDWARE_PLATFORM 1
++
++static void __eat_cpuinfo_space(char *buf)
++{
++	/* first eat trailing space */
++	char *tmp = buf + strlen(buf) - 1;
++	while (tmp > buf && isspace(*tmp))
++		*tmp-- = '\0';
++	/* then eat leading space */
++	tmp = buf;
++	while (*tmp && isspace(*tmp))
++		tmp++;
++	if (tmp != buf)
++		memmove(buf, tmp, strlen(tmp)+1);
++	/* finally collapse whitespace */
++	tmp = buf;
++	while (tmp[0] && tmp[1]) {
++		if (isspace(tmp[0]) && isspace(tmp[1])) {
++			memmove(tmp, tmp+1, strlen(tmp));
++			continue;
++		}
++		++tmp;
++	}
++}
++
++static int __linux_procinfo(int x, char *fstr, size_t s)
++{
++	FILE *fp;
++
++	char *procinfo_keys[] = {
++		/* --processor --hardware-platform */
++		#if defined(__alpha__)
++			"cpu model", "system type"
++		#elif defined(__arm__)
++			"Processor", "Hardware"
++		#elif defined(__avr32__)
++			"processor", "cpu family"
++		#elif defined(__bfin__)
++			"CPU", "BOARD Name"
++		#elif defined(__cris__)
++			"cpu", "cpu model"
++		#elif defined(__frv__)
++			"CPU-Core", "System"
++		#elif defined(__i386__) || defined(__x86_64__)
++			"model name", "vendor_id"
++		#elif defined(__ia64__)
++			"family", "vendor"
++		#elif defined(__hppa__)
++			"cpu", "model"
++		#elif defined(__m68k__)
++			"CPU", "MMU"
++		#elif defined(__mips__)
++			"cpu model", "system type"
++		#elif defined(__powerpc__) || defined(__powerpc64__)
++			"cpu", "machine"
++		#elif defined(__s390__) || defined(__s390x__)
++			"Type", "Manufacturer"
++		#elif defined(__sh__)
++			"cpu type", "machine"
++		#elif defined(sparc) || defined(__sparc__)
++			"type", "cpu"
++		#elif defined(__vax__)
++			"cpu type", "cpu"
++		#else
++			"unknown", "unknown"
++		#endif
++	};
++
++	if ((fp = fopen(CPUINFO_FILE, "r")) != NULL) {
++		char key[65], value[257], eol, *ret = NULL;
++
++		while (fscanf(fp, CPUINFO_FORMAT, key, value, &eol) != EOF) {
++			__eat_cpuinfo_space(key);
++			if (!strcmp(key, procinfo_keys[x])) {
++				__eat_cpuinfo_space(value);
++				ret = value;
++				break;
++			}
++			if (eol != '\n') {
++				/* we need two fscanf's here in case the previous
++				 * length limit caused us to read right up to the
++				 * newline ... doing "%*[^\n]\n" wont eat the newline
++				 */
++				fscanf(fp, "%*[^\n]");
++				fscanf(fp, "\n");
++			}
++		}
++		fclose(fp);
++
++		if (ret) {
++			strncpy(fstr, ret, s);
++			return 0;
++		}
++	}
++
++	return -1;
++}
++
++#endif
++
+ /* Print ELEMENT, preceded by a space if something has already been
+    printed.  */
+ 
+@@ -300,10 +416,14 @@ main (int argc, char **argv)
+   if (toprint & PRINT_PROCESSOR)
+     {
+       char const *element = unknown;
+-#if HAVE_SYSINFO && defined SI_ARCHITECTURE
++#if ( HAVE_SYSINFO && defined SI_ARCHITECTURE ) || defined(USE_PROCINFO)
+       {
+         static char processor[257];
++#if defined(USE_PROCINFO)
++        if (0 <= __linux_procinfo (PROCINFO_PROCESSOR, processor, sizeof processor))
++#else
+         if (0 <= sysinfo (SI_ARCHITECTURE, processor, sizeof processor))
++#endif
+           element = processor;
+       }
+ #endif
+@@ -356,9 +476,13 @@ main (int argc, char **argv)
+       if (element == unknown)
+         {
+           static char hardware_platform[257];
++#if defined(USE_PROCINFO)
++          if (0 <= __linux_procinfo (PROCINFO_HARDWARE_PLATFORM, hardware_platform, sizeof hardware_platform))
++#else
+           size_t s = sizeof hardware_platform;
+           static int mib[] = { CTL_HW, UNAME_HARDWARE_PLATFORM };
+           if (sysctl (mib, 2, hardware_platform, &s, 0, 0) >= 0)
++#endif
+             element = hardware_platform;
+         }
+ #endif
diff -Naurp buildroot-2017.05-rc2/package/coreutils/coreutils.mk buildroot-2017.05-rc2-patch/package/coreutils/coreutils.mk
--- buildroot-2017.05-rc2/package/coreutils/coreutils.mk	2017-05-17 10:27:16.027844496 +0200
+++ buildroot-2017.05-rc2-patch/package/coreutils/coreutils.mk	2017-09-15 16:12:28.000000000 +0200
@@ -14,46 +14,10 @@ COREUTILS_LICENSE_FILES = COPYING
 COREUTILS_AUTORECONF = YES
 COREUTILS_GETTEXTIZE = YES
 
-COREUTILS_CONF_OPTS = --disable-rpath --enable-single-binary=shebangs \
+COREUTILS_CONF_OPTS = --bindir=/bin --disable-single-binary --prefix=/usr \
 	$(if $(BR2_TOOLCHAIN_USES_MUSL),--with-included-regex)
-COREUTILS_CONF_ENV = ac_cv_c_restrict=no \
-	ac_cv_func_chown_works=yes \
-	ac_cv_func_euidaccess=no \
-	ac_cv_func_fstatat=yes \
-	ac_cv_func_getdelim=yes \
-	ac_cv_func_getgroups=yes \
-	ac_cv_func_getgroups_works=yes \
-	ac_cv_func_getloadavg=no \
-	ac_cv_func_lstat_dereferences_slashed_symlink=yes \
-	ac_cv_func_lstat_empty_string_bug=no \
-	ac_cv_func_strerror_r_char_p=no \
-	ac_cv_func_strnlen_working=yes \
-	ac_cv_func_strtod=yes \
-	ac_cv_func_working_mktime=yes \
-	ac_cv_have_decl_strerror_r=yes \
-	ac_cv_have_decl_strnlen=yes \
-	ac_cv_lib_getloadavg_getloadavg=no \
-	ac_cv_lib_util_getloadavg=no \
-	ac_fsusage_space=yes \
-	ac_use_included_regex=no \
-	am_cv_func_working_getline=yes \
-	fu_cv_sys_stat_statfs2_bsize=yes \
-	gl_cv_func_getcwd_null=yes \
-	gl_cv_func_getcwd_path_max=yes \
-	gl_cv_func_gettimeofday_clobber=no \
-	gl_cv_func_fstatat_zero_flag=no \
-	gl_cv_func_link_follows_symlink=no \
-	gl_cv_func_re_compile_pattern_working=yes \
-	gl_cv_func_svid_putenv=yes \
-	gl_cv_func_tzset_clobber=no \
-	gl_cv_func_working_mkstemp=yes \
-	gl_cv_func_working_utimes=yes \
-	gl_getline_needs_run_time_check=no \
-	gl_cv_have_proc_uptime=yes \
-	utils_cv_localtime_cache=no \
-	PERL=missing \
-	MAKEINFO=true \
-	INSTALL_PROGRAM=$(INSTALL)
+
+COREUTILS_CONF_ENV += FORCE_UNSAFE_CONFIGURE=1
 
 COREUTILS_BIN_PROGS = cat chgrp chmod chown cp date dd df dir echo false \
 	ln ls mkdir mknod mv pwd rm rmdir vdir sleep stty sync touch true \
diff -Naurp buildroot-2017.05-rc2/package/dbus/dbus.mk buildroot-2017.05-rc2-patch/package/dbus/dbus.mk
--- buildroot-2017.05-rc2/package/dbus/dbus.mk	2017-05-17 10:27:16.027844496 +0200
+++ buildroot-2017.05-rc2-patch/package/dbus/dbus.mk	2017-09-15 16:08:41.000000000 +0200
@@ -77,26 +77,11 @@ else
 DBUS_CONF_OPTS += --disable-systemd
 endif
 
-# fix rebuild (dbus makefile errors out if /var/lib/dbus is a symlink)
-define DBUS_REMOVE_VAR_LIB_DBUS
-	rm -rf $(TARGET_DIR)/var/lib/dbus
-endef
-
-DBUS_PRE_INSTALL_TARGET_HOOKS += DBUS_REMOVE_VAR_LIB_DBUS
-
-define DBUS_REMOVE_DEVFILES
-	rm -rf $(TARGET_DIR)/usr/lib/dbus-1.0
-endef
-
-DBUS_POST_INSTALL_TARGET_HOOKS += DBUS_REMOVE_DEVFILES
-
 define DBUS_INSTALL_INIT_SYSV
 	$(INSTALL) -m 0755 -D package/dbus/S30dbus \
 		$(TARGET_DIR)/etc/init.d/S30dbus
 
-	mkdir -p $(TARGET_DIR)/var/lib
-	rm -rf $(TARGET_DIR)/var/lib/dbus
-	ln -sf /tmp/dbus $(TARGET_DIR)/var/lib/dbus
+	mkdir -p $(TARGET_DIR)/var/lib/dbus
 endef
 
 define DBUS_INSTALL_INIT_SYSTEMD
diff -Naurp buildroot-2017.05-rc2/package/dbus-glib/dbus-glib.mk buildroot-2017.05-rc2-patch/package/dbus-glib/dbus-glib.mk
--- buildroot-2017.05-rc2/package/dbus-glib/dbus-glib.mk	2017-05-17 10:27:16.027844496 +0200
+++ buildroot-2017.05-rc2-patch/package/dbus-glib/dbus-glib.mk	2017-09-15 16:20:18.000000000 +0200
@@ -19,7 +19,6 @@ DBUS_GLIB_CONF_OPTS = \
 	--disable-tests \
 	--disable-xml-docs \
 	--with-introspect-xml=$(DBUS_HOST_INTROSPECT) \
-	--with-dbus-binding-tool=$(DBUS_GLIB_HOST_BINARY) \
 	--disable-bash-completion \
 	--disable-doxygen-docs \
 	--enable-asserts=yes
diff -Naurp buildroot-2017.05-rc2/package/e2fsprogs/e2fsprogs.mk buildroot-2017.05-rc2-patch/package/e2fsprogs/e2fsprogs.mk
--- buildroot-2017.05-rc2/package/e2fsprogs/e2fsprogs.mk	2017-05-17 10:27:16.027844496 +0200
+++ buildroot-2017.05-rc2-patch/package/e2fsprogs/e2fsprogs.mk	2017-06-19 10:07:16.000000000 +0200
@@ -9,95 +9,28 @@ E2FSPROGS_SOURCE = e2fsprogs-$(E2FSPROGS
 E2FSPROGS_SITE = $(BR2_KERNEL_MIRROR)/linux/kernel/people/tytso/e2fsprogs/v$(E2FSPROGS_VERSION)
 E2FSPROGS_LICENSE = GPL-2.0, MIT-like with advertising clause (libss and libet)
 E2FSPROGS_LICENSE_FILES = NOTICE lib/ss/mit-sipb-copyright.h lib/et/internal.h
-E2FSPROGS_INSTALL_STAGING = YES
+E2FSPROGS_INSTALL_STAGING = NO
 
-# Use libblkid and libuuid from util-linux for host and target packages.
-# This prevents overriding them with e2fsprogs' ones, which may cause
-# problems for other packages.
-E2FSPROGS_DEPENDENCIES = host-pkgconf util-linux
-HOST_E2FSPROGS_DEPENDENCIES = host-pkgconf host-util-linux
-
-# If both e2fsprogs and busybox are selected, make certain e2fsprogs
-# wins the fight over who gets to have their utils actually installed
-ifeq ($(BR2_PACKAGE_BUSYBOX),y)
-E2FSPROGS_DEPENDENCIES += busybox
-endif
-
-# e4defrag doesn't build on older systems like RHEL5.x, and we don't
-# need it on the host anyway.
-# Disable fuse2fs as well to avoid carrying over deps, and it's unused
-HOST_E2FSPROGS_CONF_OPTS = \
-	--disable-defrag \
-	--disable-e2initrd-helper \
-	--disable-fuse2fs \
-	--disable-libblkid \
-	--disable-libuuid \
-	--enable-symlink-install \
-	--disable-testio-debug
+E2FSPROGS_DEPENDENCIES = pkgconf util-linux
+E2FSPROGS_DEPENDENCIES += libfuse
 
-# Set the binary directories to "/bin" and "/sbin" to override programs
-# installed by busybox.
 E2FSPROGS_CONF_OPTS = \
+	--prefix=/usr \
+	--with-root-prefix="" \
 	--bindir=/bin \
 	--sbindir=/sbin \
-	$(if $(BR2_STATIC_LIBS),--disable-elf-shlibs,--enable-elf-shlibs) \
-	$(if $(BR2_PACKAGE_E2FSPROGS_DEBUGFS),--enable-debugfs,--disable-debugfs) \
-	$(if $(BR2_PACKAGE_E2FSPROGS_E2IMAGE),--enable-imager,--disable-imager) \
-	$(if $(BR2_PACKAGE_E2FSPROGS_E4DEFRAG),--enable-defrag,--disable-defrag) \
-	$(if $(BR2_PACKAGE_E2FSPROGS_FSCK),--enable-fsck,--disable-fsck) \
-	$(if $(BR2_PACKAGE_E2FSPROGS_RESIZE2FS),--enable-resizer,--disable-resizer) \
+	--enable-elf-shlibs \
+	--enable-fsck \
+	--enable-fuse2fs \
 	--disable-uuidd \
 	--disable-libblkid \
 	--disable-libuuid \
-	--disable-e2initrd-helper \
 	--disable-testio-debug \
-	--disable-rpath \
 	--enable-symlink-install
 
-ifeq ($(BR2_PACKAGE_E2FSPROGS_FUSE2FS),y)
-E2FSPROGS_CONF_OPTS += --enable-fuse2fs
-E2FSPROGS_DEPENDENCIES += libfuse
-else
-E2FSPROGS_CONF_OPTS += --disable-fuse2fs
-endif
-
-ifeq ($(BR2_nios2),y)
-E2FSPROGS_CONF_ENV += ac_cv_func_fallocate=no
-endif
-
-# Some programs are built for the host, but use definitions guessed by
-# the configure script (i.e with the cross-compiler). Help them by
-# saying that <sys/stat.h> is available on the host, which is needed
-# for util/subst.c to build properly.
-E2FSPROGS_CONF_ENV += BUILD_CFLAGS="-DHAVE_SYS_STAT_H"
-
-# Disable use of the host magic.h, as on older hosts (e.g. RHEL 5)
-# it doesn't provide definitions expected by e2fsprogs support lib.
-HOST_E2FSPROGS_CONF_ENV += \
-	ac_cv_header_magic_h=no \
-	ac_cv_lib_magic_magic_file=no
-
-ifeq ($(BR2_NEEDS_GETTEXT_IF_LOCALE)$(BR2_STATIC_LIBS),yy)
-# util-linux libuuid pulls in libintl if needed, so ensure we also
-# link against it, otherwise static linking fails
-E2FSPROGS_CONF_ENV += LIBS=-lintl
-endif
-
-E2FSPROGS_MAKE_OPTS = LDCONFIG=true
-
-E2FSPROGS_INSTALL_STAGING_OPTS = \
-	DESTDIR=$(STAGING_DIR) \
-	LDCONFIG=true \
-	install-libs
-
-E2FSPROGS_INSTALL_TARGET_OPTS = \
-	DESTDIR=$(TARGET_DIR) \
-	LDCONFIG=true \
-	install
-
-define HOST_E2FSPROGS_INSTALL_CMDS
-	$(HOST_MAKE_ENV) $(MAKE) -C $(@D) install install-libs
+define E2FSPROGS_INSTALL_TARGET_CMDS
+	$(MAKE_ENV) $(MAKE) -C $(@D) DESTDIR=$(TARGET_DIR) install
+	$(MAKE_ENV) $(MAKE) -C $(@D) DESTDIR=$(TARGET_DIR) install-libs
 endef
 
 $(eval $(autotools-package))
-$(eval $(host-autotools-package))
diff -Naurp buildroot-2017.05-rc2/package/freetype/freetype.mk buildroot-2017.05-rc2-patch/package/freetype/freetype.mk
--- buildroot-2017.05-rc2/package/freetype/freetype.mk	2017-05-17 10:27:16.027844496 +0200
+++ buildroot-2017.05-rc2-patch/package/freetype/freetype.mk	2017-09-15 16:08:41.000000000 +0200
@@ -22,6 +22,7 @@ HOST_FREETYPE_CONF_OPTS = --without-zlib
 # autogen.sh is because autotools stuff lives in other directories and
 # even AUTORECONF with _OPTS doesn't do it properly.
 # POST_PATCH is because we still need to patch libtool after the regen.
+FREETYPE_LIBTOOL_PATCH = NO
 define FREETYPE_RUN_AUTOGEN
 	cd $(@D) && PATH=$(BR_PATH) ./autogen.sh
 endef
diff -Naurp buildroot-2017.05-rc2/package/getent/getent.mk buildroot-2017.05-rc2-patch/package/getent/getent.mk
--- buildroot-2017.05-rc2/package/getent/getent.mk	2017-05-17 10:27:16.027844496 +0200
+++ buildroot-2017.05-rc2-patch/package/getent/getent.mk	2017-09-15 16:08:41.000000000 +0200
@@ -9,15 +9,7 @@ GETENT_LICENSE = LGPL-2.1+
 # For glibc toolchains, we use the getent program built/installed by
 # the C library. For other toolchains, we use the wrapper script
 # included in this package.
-ifeq ($(BR2_TOOLCHAIN_USES_GLIBC),y)
-# Sourcery toolchains install it in sysroot/usr/lib/bin
-# Buildroot toolchains install it in sysroot/usr/bin
-GETENT_LOCATION = $(firstword $(wildcard \
-	$(STAGING_DIR)/usr/bin/getent \
-	$(STAGING_DIR)/usr/lib/bin/getent))
-else
 GETENT_LOCATION = package/getent/getent
-endif
 
 define GETENT_INSTALL_TARGET_CMDS
 	$(INSTALL) -D -m 0755 $(GETENT_LOCATION) $(TARGET_DIR)/usr/bin/getent
diff -Naurp buildroot-2017.05-rc2/package/gettext/gettext.mk buildroot-2017.05-rc2-patch/package/gettext/gettext.mk
--- buildroot-2017.05-rc2/package/gettext/gettext.mk	2017-05-17 10:27:16.027844496 +0200
+++ buildroot-2017.05-rc2-patch/package/gettext/gettext.mk	2017-09-15 16:08:41.000000000 +0200
@@ -16,15 +16,24 @@ GETTEXT_DEPENDENCIES = $(if $(BR2_PACKAG
 # Avoid using the bundled subset of libxml2
 HOST_GETTEXT_DEPENDENCIES = host-libxml2
 
+#GETTEXT_AUTORECONF = YES
+
+#GETTEXT_CONF_OPTS += \
+#	--disable-libasprintf \
+#	--disable-acl \
+#	--disable-openmp \
+#	--disable-rpath \
+#	--disable-java \
+#	--disable-native-java \
+#	--disable-csharp \
+#	--disable-relocatable \
+#	--without-emacs
+
 GETTEXT_CONF_OPTS += \
-	--disable-libasprintf \
 	--disable-acl \
-	--disable-openmp \
-	--disable-rpath \
 	--disable-java \
 	--disable-native-java \
 	--disable-csharp \
-	--disable-relocatable \
 	--without-emacs
 
 HOST_GETTEXT_CONF_OPTS = \
@@ -40,7 +49,7 @@ HOST_GETTEXT_CONF_OPTS = \
 
 # For the target version, we only need the runtime, and for the host
 # version, we only need the tools.
-GETTEXT_SUBDIR = gettext-runtime
+#GETTEXT_SUBDIR = gettext-runtime
 HOST_GETTEXT_SUBDIR = gettext-tools
 
 # Disable the build of documentation and examples of gettext-tools,
@@ -53,12 +62,12 @@ endef
 GETTEXT_POST_PATCH_HOOKS += HOST_GETTEXT_DISABLE_UNNEEDED
 HOST_GETTEXT_POST_PATCH_HOOKS += HOST_GETTEXT_DISABLE_UNNEEDED
 
-define GETTEXT_REMOVE_UNNEEDED
-	$(RM) -rf $(TARGET_DIR)/usr/share/gettext/ABOUT-NLS
-	rmdir --ignore-fail-on-non-empty $(TARGET_DIR)/usr/share/gettext
-endef
+#define GETTEXT_REMOVE_UNNEEDED
+#	$(RM) -rf $(TARGET_DIR)/usr/share/gettext/ABOUT-NLS
+#	rmdir --ignore-fail-on-non-empty $(TARGET_DIR)/usr/share/gettext
+#endef
 
-GETTEXT_POST_INSTALL_TARGET_HOOKS += GETTEXT_REMOVE_UNNEEDED
+#GETTEXT_POST_INSTALL_TARGET_HOOKS += GETTEXT_REMOVE_UNNEEDED
 
 # Force build with NLS support, otherwise libintl is not built
 # This is needed because some packages (eg. libglib2) requires
@@ -85,6 +94,13 @@ endef
 
 HOST_GETTEXT_POST_INSTALL_HOOKS += HOST_GETTEXT_ADD_ABOUT_NLS
 
+define GETTEXT_BUILD_CMDS
+	+$(shell find $(@D) -type f -name Makefile | xargs sed -i 's%-L/usr/lib%-Wl,-rpath-link,$(STAGING_DIR)/usr/lib%g')
+	+$(shell find $(@D) -type f -name Makefile | xargs sed -i 's%/usr/lib/libunistring.so%-Wl,-rpath-link,$(STAGING_DIR)/usr/lib -lunistring%g')
+	+$(shell find $(@D) -type f -name Makefile | xargs sed -i 's%/usr/lib/libncurses.so%-Wl,-rpath-link,$(STAGING_DIR)/usr/lib -lncurses%g')
+	$(TARGET_MAKE_ENV) $(TARGET_CONFIGURE_ARGS) $(MAKE) $(GETTEXT_MAKE_OPTS) -C $(@D)
+endef
+
 GETTEXTIZE = $(HOST_CONFIGURE_OPTS) AUTOM4TE=$(HOST_DIR)/usr/bin/autom4te $(HOST_DIR)/usr/bin/gettextize -f
 
 $(eval $(autotools-package))
diff -Naurp buildroot-2017.05-rc2/package/glibc/glibc.mk buildroot-2017.05-rc2-patch/package/glibc/glibc.mk
--- buildroot-2017.05-rc2/package/glibc/glibc.mk	2017-05-17 10:27:16.027844496 +0200
+++ buildroot-2017.05-rc2-patch/package/glibc/glibc.mk	2017-09-15 16:08:41.000000000 +0200
@@ -99,21 +99,6 @@ endef
 # to install the libraries, and nothing more.
 #
 
-GLIBC_LIBS_LIB = \
-	ld*.so.* libanl.so.* libc.so.* libcrypt.so.* libdl.so.* libgcc_s.so.* \
-	libm.so.* libnsl.so.* libpthread.so.* libresolv.so.* librt.so.* \
-	libutil.so.* libnss_files.so.* libnss_dns.so.* libmvec.so.*
-
-ifeq ($(BR2_PACKAGE_GDB),y)
-GLIBC_LIBS_LIB += libthread_db.so.*
-endif
-
-define GLIBC_INSTALL_TARGET_CMDS
-	for libpattern in $(GLIBC_LIBS_LIB); do \
-		$(call copy_toolchain_lib_root,$$libpattern) ; \
-	done
-endef
-
 # MIPS R6 requires to have NaN2008 support which is currently not
 # supported by the Linux kernel. In order to prevent building the
 # glibc against kernels not having NaN2008 support on platforms that
diff -Naurp buildroot-2017.05-rc2/package/gptfdisk/gptfdisk.hash buildroot-2017.05-rc2-patch/package/gptfdisk/gptfdisk.hash
--- buildroot-2017.05-rc2/package/gptfdisk/gptfdisk.hash	2017-05-17 10:27:16.027844496 +0200
+++ buildroot-2017.05-rc2-patch/package/gptfdisk/gptfdisk.hash	2017-09-15 16:08:41.000000000 +0200
@@ -1,3 +1,2 @@
 # From http://sourceforge.net/projects/gptfdisk/files/gptfdisk/1.0.0/
-md5	2061f917af084215898d4fea04d8388f	gptfdisk-1.0.0.tar.gz
-sha1	97f25db2def1c1939ed3954520bfd9948806d95a	gptfdisk-1.0.0.tar.gz
+sha1	ad28c511c642235815b83fffddf728c117057cba	gptfdisk-1.0.1.tar.gz
diff -Naurp buildroot-2017.05-rc2/package/gptfdisk/gptfdisk.mk buildroot-2017.05-rc2-patch/package/gptfdisk/gptfdisk.mk
--- buildroot-2017.05-rc2/package/gptfdisk/gptfdisk.mk	2017-05-17 10:27:16.027844496 +0200
+++ buildroot-2017.05-rc2-patch/package/gptfdisk/gptfdisk.mk	2017-09-15 16:08:41.000000000 +0200
@@ -4,7 +4,7 @@
 #
 ################################################################################
 
-GPTFDISK_VERSION = 1.0.0
+GPTFDISK_VERSION = 1.0.1
 GPTFDISK_SITE = http://downloads.sourceforge.net/sourceforge/gptfdisk
 GPTFDISK_LICENSE = GPL-2.0+
 GPTFDISK_LICENSE_FILES = COPYING
diff -Naurp buildroot-2017.05-rc2/package/groff/Config.in buildroot-2017.05-rc2-patch/package/groff/Config.in
--- buildroot-2017.05-rc2/package/groff/Config.in	1970-01-01 01:00:00.000000000 +0100
+++ buildroot-2017.05-rc2-patch/package/groff/Config.in	2017-05-28 22:07:29.000000000 +0200
@@ -0,0 +1,6 @@
+config BR2_PACKAGE_GROFF
+	bool "groff"
+	help
+	  groff
+
+	  https://......
diff -Naurp buildroot-2017.05-rc2/package/groff/groff.hash buildroot-2017.05-rc2-patch/package/groff/groff.hash
--- buildroot-2017.05-rc2/package/groff/groff.hash	1970-01-01 01:00:00.000000000 +0100
+++ buildroot-2017.05-rc2-patch/package/groff/groff.hash	2017-05-28 22:06:51.000000000 +0200
@@ -0,0 +1,2 @@
+# Locally calculated after checking pgp signature
+sha256	3a48a9d6c97750bfbd535feeb5be0111db6406ddb7bb79fc680809cda6d828a5	groff-1.22.3.tar.gz
diff -Naurp buildroot-2017.05-rc2/package/groff/groff.mk buildroot-2017.05-rc2-patch/package/groff/groff.mk
--- buildroot-2017.05-rc2/package/groff/groff.mk	1970-01-01 01:00:00.000000000 +0100
+++ buildroot-2017.05-rc2-patch/package/groff/groff.mk	2017-05-28 22:15:10.000000000 +0200
@@ -0,0 +1,14 @@
+################################################################################
+#
+# groff
+#
+################################################################################
+
+GROFF_VERSION = 1.22.3
+GROFF_SOURCE = groff-$(GROFF_VERSION).tar.gz
+GROFF_SITE = https://ftp.osuosl.org/pub/lfs/lfs-packages/8.0
+GROFF_LICENSE = GPL-3.0+
+GROFF_LICENSE_FILES = COPYING
+GROFF_CONF_OPTS = --without-x
+
+$(eval $(autotools-package))
diff -Naurp buildroot-2017.05-rc2/package/heimdal/Config.in buildroot-2017.05-rc2-patch/package/heimdal/Config.in
--- buildroot-2017.05-rc2/package/heimdal/Config.in	1970-01-01 01:00:00.000000000 +0100
+++ buildroot-2017.05-rc2-patch/package/heimdal/Config.in	2017-09-15 16:08:41.000000000 +0200
@@ -0,0 +1,6 @@
+config BR2_PACKAGE_HEIMDAL
+	bool "heimdal"
+	help
+	  heimdal
+
+	  https://......
diff -Naurp buildroot-2017.05-rc2/package/heimdal/heimdal.mk buildroot-2017.05-rc2-patch/package/heimdal/heimdal.mk
--- buildroot-2017.05-rc2/package/heimdal/heimdal.mk	2017-05-17 10:27:16.027844496 +0200
+++ buildroot-2017.05-rc2-patch/package/heimdal/heimdal.mk	2017-09-15 16:08:41.000000000 +0200
@@ -6,8 +6,10 @@
 
 HEIMDAL_VERSION = 7.1.0
 HEIMDAL_SITE = http://www.h5l.org/dist/src
-HOST_HEIMDAL_DEPENDENCIES = host-e2fsprogs host-ncurses host-pkgconf
+HOST_HEIMDAL_DEPENDENCIES = host-e2fsprogs host-ncurses pkgconf
+HEIMDAL_DEPENDENCIES = e2fsprogs ncurses pkgconf
 HEIMDAL_INSTALL_STAGING = YES
+HEIMDAL_LIBTOOL_PATCH = NO
 HEIMDAL_MAKE = $(MAKE1)
 # static because of -fPIC issues with e2fsprogs on x86_64 host
 HOST_HEIMDAL_CONF_OPTS = \
@@ -24,6 +26,21 @@ HOST_HEIMDAL_CONF_OPTS = \
 	--without-hesiod \
 	--without-x \
 	--disable-heimdal-documentation
+HEIMDAL_CONF_OPTS = \
+	--libdir=/usr/lib/heimdal \
+	--includedir=/usr/include/heimdal \
+	--without-openldap \
+	--without-capng \
+	--without-sqlite3 \
+	--without-libintl \
+	--without-openssl \
+	--without-berkeley-db \
+	--without-readline \
+	--without-libedit \
+	--without-hesiod \
+	--without-x \
+	--disable-heimdal-documentation
+
 HOST_HEIMDAL_CONF_ENV = MAKEINFO=true
 HEIMDAL_LICENSE = BSD-3-Clause
 HEIMDAL_LICENSE_FILES = LICENSE
@@ -38,4 +55,14 @@ endef
 
 HOST_HEIMDAL_POST_INSTALL_HOOKS += HOST_HEIMDAL_MAKE_SYMLINK
 
+define HEIMDAL_MAKE_SYMLINK
+	ln -sf $(TARGET_DIR)/usr/libexec/heimdal/asn1_compile \
+		$(TARGET_DIR)/usr/bin/asn1_compile
+	ln -sf $(TARGET_DIR)/usr/bin/compile_et \
+		$(TARGET_DIR)/usr/libexec/heimdal/compile_et
+endef
+
+HEIMDAL_POST_INSTALL_HOOKS += HEIMDAL_MAKE_SYMLINK
+
+$(eval $(autotools-package))
 $(eval $(host-autotools-package))
diff -Naurp buildroot-2017.05-rc2/package/iw/iw-4.9-dont_gz_manpage.patch buildroot-2017.05-rc2-patch/package/iw/iw-4.9-dont_gz_manpage.patch
--- buildroot-2017.05-rc2/package/iw/iw-4.9-dont_gz_manpage.patch	1970-01-01 01:00:00.000000000 +0100
+++ buildroot-2017.05-rc2-patch/package/iw/iw-4.9-dont_gz_manpage.patch	2017-09-15 16:08:41.000000000 +0200
@@ -0,0 +1,19 @@
+diff -Naur iw-4.9/Makefile iw-4.9-dont_gz_manpage/Makefile
+--- iw-4.9/Makefile	2016-10-18 05:46:20.029451264 +0000
++++ iw-4.9-dont_gz_manpage/Makefile	2017-05-25 06:27:40.462803650 +0000
+@@ -115,13 +115,13 @@
+ 	@$(NQ) ' GZIP' $<
+ 	$(Q)gzip < $< > $@
+ 
+-install: iw iw.8.gz
++install: iw iw.8
+ 	@$(NQ) ' INST iw'
+ 	$(Q)$(MKDIR) $(DESTDIR)$(SBINDIR)
+ 	$(Q)$(INSTALL) -m 755 iw $(DESTDIR)$(SBINDIR)
+ 	@$(NQ) ' INST iw.8'
+ 	$(Q)$(MKDIR) $(DESTDIR)$(MANDIR)/man8/
+-	$(Q)$(INSTALL) -m 644 iw.8.gz $(DESTDIR)$(MANDIR)/man8/
++	$(Q)$(INSTALL) -m 644 iw.8 $(DESTDIR)$(MANDIR)/man8/
+ 
+ clean:
+ 	$(Q)rm -f iw *.o *~ *.gz version.c *-stamp
diff -Naurp buildroot-2017.05-rc2/package/joe/Config.in buildroot-2017.05-rc2-patch/package/joe/Config.in
--- buildroot-2017.05-rc2/package/joe/Config.in	2017-05-17 10:27:16.027844496 +0200
+++ buildroot-2017.05-rc2-patch/package/joe/Config.in	2017-02-28 22:00:23.000000000 +0100
@@ -13,8 +13,8 @@ config BR2_PACKAGE_JOE
 
 config BR2_PACKAGE_JOE_FULL
 	bool "full install of rc and syntax highlighting files"
-	default y
 	depends on BR2_PACKAGE_JOE
+	default y
 	help
 	  Install all rc and syntax highlighting files. Without this option
 	  only the joe binary is installed, which contains a default
diff -Naurp buildroot-2017.05-rc2/package/joe/joe.hash buildroot-2017.05-rc2-patch/package/joe/joe.hash
--- buildroot-2017.05-rc2/package/joe/joe.hash	2017-05-17 10:27:16.027844496 +0200
+++ buildroot-2017.05-rc2-patch/package/joe/joe.hash	2017-05-17 18:03:53.000000000 +0200
@@ -1,2 +1,2 @@
 # Locally computed:
-sha256  cae456e1ad5a8c1d3c94920a3416c2347277739b260e3494d3bc0f2b9b73106f  joe-3.7.tar.gz
+sha256  a5704828bbca29acb9e200414fef522c66cdf9ce28150f402d6767da43460979  joe-4.4.tar.gz
diff -Naurp buildroot-2017.05-rc2/package/joe/joe.mk buildroot-2017.05-rc2-patch/package/joe/joe.mk
--- buildroot-2017.05-rc2/package/joe/joe.mk	2017-05-17 10:27:16.027844496 +0200
+++ buildroot-2017.05-rc2-patch/package/joe/joe.mk	2017-05-17 18:02:41.000000000 +0200
@@ -4,9 +4,9 @@
 #
 ################################################################################
 
-JOE_VERSION = 3.7
-JOE_SITE = http://downloads.sourceforge.net/project/joe-editor/JOE%20sources/joe-$(JOE_VERSION)
-JOE_LICENSE = GPL-1.0+
+JOE_VERSION = 4.4
+JOE_SITE = https://ftp.osuosl.org/pub/blfs/conglomeration/joe
+JOE_LICENSE = GPLv1+
 JOE_LICENSE_FILES = COPYING
 
 ifeq ($(BR2_PACKAGE_NCURSES),y)
diff -Naurp buildroot-2017.05-rc2/package/lftp/lftp.mk buildroot-2017.05-rc2-patch/package/lftp/lftp.mk
--- buildroot-2017.05-rc2/package/lftp/lftp.mk	2017-05-17 10:27:16.027844496 +0200
+++ buildroot-2017.05-rc2-patch/package/lftp/lftp.mk	2017-09-12 21:38:26.000000000 +0200
@@ -12,12 +12,8 @@ LFTP_LICENSE_FILES = COPYING
 # Needed so that our libtool patch applies properly, and for patch
 # 0001-fix-static-link-with-readline.patch.
 LFTP_AUTORECONF = YES
-LFTP_DEPENDENCIES = readline zlib host-pkgconf
-
-# Help lftp finding readline and zlib
-LFTP_CONF_OPTS = \
-	--with-readline=$(STAGING_DIR)/usr \
-	--with-zlib=$(STAGING_DIR)/usr
+LFTP_LIBTOOL_PATCH = NO
+LFTP_DEPENDENCIES = readline zlib pkgconf
 
 ifneq ($(BR2_STATIC_LIBS),y)
 LFTP_CONF_OPTS += --with-modules
@@ -42,27 +38,27 @@ LFTP_CONF_OPTS += --without-openssl
 endif
 
 # Remove /usr/share/lftp
-define LFTP_REMOVE_DATA
-	$(RM) -fr $(TARGET_DIR)/usr/share/lftp
-endef
+#define LFTP_REMOVE_DATA
+#	$(RM) -fr $(TARGET_DIR)/usr/share/lftp
+#endef
 
-LFTP_POST_INSTALL_TARGET_HOOKS += LFTP_REMOVE_DATA
+#LFTP_POST_INSTALL_TARGET_HOOKS += LFTP_REMOVE_DATA
 
 # Optional commands and protocols
-LFTP_MODULES_TO_REMOVE-$(BR2_PACKAGE_LFTP_CMD_MIRROR) += cmd-mirror.so
-LFTP_MODULES_TO_REMOVE-$(BR2_PACKAGE_LFTP_CMD_SLEEP) += cmd-sleep.so
-LFTP_MODULES_TO_REMOVE-$(BR2_PACKAGE_LFTP_CMD_TORRENT) += cmd-torrent.so
-LFTP_MODULES_TO_REMOVE-$(BR2_PACKAGE_LFTP_PROTO_FISH) += proto-fish.so
-LFTP_MODULES_TO_REMOVE-$(BR2_PACKAGE_LFTP_PROTO_FTP) += proto-ftp.so
-LFTP_MODULES_TO_REMOVE-$(BR2_PACKAGE_LFTP_PROTO_HTTP) += proto-http.so
-LFTP_MODULES_TO_REMOVE-$(BR2_PACKAGE_LFTP_PROTO_SFTP) += proto-sftp.so
-
-define LFTP_REMOVE_MODULES
-	for f in $(LFTP_MODULES_TO_REMOVE-) ; do \
-		$(RM) -f $(TARGET_DIR)/usr/lib/lftp/$(LFTP_VERSION)/$$f ; \
-	done
-endef
+#LFTP_MODULES_TO_REMOVE-$(BR2_PACKAGE_LFTP_CMD_MIRROR) += cmd-mirror.so
+#LFTP_MODULES_TO_REMOVE-$(BR2_PACKAGE_LFTP_CMD_SLEEP) += cmd-sleep.so
+#LFTP_MODULES_TO_REMOVE-$(BR2_PACKAGE_LFTP_CMD_TORRENT) += cmd-torrent.so
+#LFTP_MODULES_TO_REMOVE-$(BR2_PACKAGE_LFTP_PROTO_FISH) += proto-fish.so
+#LFTP_MODULES_TO_REMOVE-$(BR2_PACKAGE_LFTP_PROTO_FTP) += proto-ftp.so
+#LFTP_MODULES_TO_REMOVE-$(BR2_PACKAGE_LFTP_PROTO_HTTP) += proto-http.so
+#LFTP_MODULES_TO_REMOVE-$(BR2_PACKAGE_LFTP_PROTO_SFTP) += proto-sftp.so
+
+#define LFTP_REMOVE_MODULES
+#	for f in $(LFTP_MODULES_TO_REMOVE-) ; do \
+#		$(RM) -f $(TARGET_DIR)/usr/lib/lftp/$(LFTP_VERSION)/$$f ; \
+#	done
+#endef
 
-LFTP_POST_INSTALL_TARGET_HOOKS += LFTP_REMOVE_MODULES
+#LFTP_POST_INSTALL_TARGET_HOOKS += LFTP_REMOVE_MODULES
 
 $(eval $(autotools-package))
diff -Naurp buildroot-2017.05-rc2/package/libfuse/libfuse.mk buildroot-2017.05-rc2-patch/package/libfuse/libfuse.mk
--- buildroot-2017.05-rc2/package/libfuse/libfuse.mk	2017-05-17 10:27:16.027844496 +0200
+++ buildroot-2017.05-rc2-patch/package/libfuse/libfuse.mk	2017-09-15 16:08:41.000000000 +0200
@@ -16,9 +16,4 @@ LIBFUSE_CONF_OPTS = \
 	--enable-lib \
 	--enable-util
 
-define LIBFUSE_INSTALL_TARGET_CMDS
-	cp -dpf $(STAGING_DIR)/usr/bin/fusermount $(TARGET_DIR)/usr/bin/
-	cp -dpf $(STAGING_DIR)/usr/lib/libfuse.so* $(TARGET_DIR)/usr/lib/
-endef
-
 $(eval $(autotools-package))
diff -Naurp buildroot-2017.05-rc2/package/libinput/libinput.mk buildroot-2017.05-rc2-patch/package/libinput/libinput.mk
--- buildroot-2017.05-rc2/package/libinput/libinput.mk	2017-05-17 10:27:16.027844496 +0200
+++ buildroot-2017.05-rc2-patch/package/libinput/libinput.mk	2017-05-27 17:13:31.000000000 +0200
@@ -12,7 +12,7 @@ LIBINPUT_INSTALL_STAGING = YES
 LIBINPUT_LICENSE = MIT
 LIBINPUT_LICENSE_FILES = COPYING
 # Tests need fork, so just disable them everywhere.
-LIBINPUT_CONF_OPTS = --disable-tests --disable-libwacom
+LIBINPUT_CONF_OPTS = --disable-tests --disable-libwacom --with-udev-dir=/lib/udev
 
 ifeq ($(BR2_PACKAGE_LIBGTK3),y)
 LIBINPUT_CONF_OPTS += --enable-event-gui
diff -Naurp buildroot-2017.05-rc2/package/libnl/libnl.mk buildroot-2017.05-rc2-patch/package/libnl/libnl.mk
--- buildroot-2017.05-rc2/package/libnl/libnl.mk	2017-05-17 10:27:16.027844496 +0200
+++ buildroot-2017.05-rc2-patch/package/libnl/libnl.mk	2017-09-15 16:08:41.000000000 +0200
@@ -12,6 +12,7 @@ LIBNL_INSTALL_STAGING = YES
 LIBNL_DEPENDENCIES = host-bison host-flex host-pkgconf
 # Patching Makefile.am
 LIBNL_AUTORECONF = YES
+LIBNL_LIBTOOL_PATCH = NO
 
 ifeq ($(BR2_PACKAGE_LIBNL_TOOLS),y)
 LIBNL_CONF_OPTS += --enable-cli
diff -Naurp buildroot-2017.05-rc2/package/libpipeline/Config.in buildroot-2017.05-rc2-patch/package/libpipeline/Config.in
--- buildroot-2017.05-rc2/package/libpipeline/Config.in	1970-01-01 01:00:00.000000000 +0100
+++ buildroot-2017.05-rc2-patch/package/libpipeline/Config.in	2017-05-28 21:50:10.000000000 +0200
@@ -0,0 +1,6 @@
+config BR2_PACKAGE_LIBPIPELINE
+	bool "libpipeline"
+	help
+	  libpipeline
+
+	  https://......
diff -Naurp buildroot-2017.05-rc2/package/libpipeline/libpipeline.hash buildroot-2017.05-rc2-patch/package/libpipeline/libpipeline.hash
--- buildroot-2017.05-rc2/package/libpipeline/libpipeline.hash	1970-01-01 01:00:00.000000000 +0100
+++ buildroot-2017.05-rc2-patch/package/libpipeline/libpipeline.hash	2017-05-28 21:52:41.000000000 +0200
@@ -0,0 +1,2 @@
+# Locally calculated after checking pgp signature
+sha256	da46d7b20163aadb9db2faae483f734e9096a7550c84b94029abeab62dd1b9ee	libpipeline-1.4.1.tar.gz
diff -Naurp buildroot-2017.05-rc2/package/libpipeline/libpipeline.mk buildroot-2017.05-rc2-patch/package/libpipeline/libpipeline.mk
--- buildroot-2017.05-rc2/package/libpipeline/libpipeline.mk	1970-01-01 01:00:00.000000000 +0100
+++ buildroot-2017.05-rc2-patch/package/libpipeline/libpipeline.mk	2017-05-28 21:51:44.000000000 +0200
@@ -0,0 +1,14 @@
+################################################################################
+#
+# libpipeline
+#
+################################################################################
+
+LIBPIPELINE_VERSION = 1.4.1
+LIBPIPELINE_SOURCE = libpipeline-$(LIBPIPELINE_VERSION).tar.gz
+LIBPIPELINE_SITE = https://ftp.osuosl.org/pub/lfs/lfs-packages/8.0
+LIBPIPELINE_LICENSE = GPL-3.0+
+LIBPIPELINE_LICENSE_FILES = COPYING
+LIBPIPELINE_DEPENDENCIES = libpipeline
+
+$(eval $(autotools-package))
diff -Naurp buildroot-2017.05-rc2/package/libpng/libpng.mk buildroot-2017.05-rc2-patch/package/libpng/libpng.mk
--- buildroot-2017.05-rc2/package/libpng/libpng.mk	2017-05-17 10:27:16.027844496 +0200
+++ buildroot-2017.05-rc2-patch/package/libpng/libpng.mk	2017-09-15 19:04:30.000000000 +0200
@@ -15,6 +15,8 @@ LIBPNG_DEPENDENCIES = host-pkgconf zlib
 HOST_LIBPNG_DEPENDENCIES = host-pkgconf host-zlib
 LIBPNG_CONFIG_SCRIPTS = libpng$(LIBPNG_SERIES)-config libpng-config
 
+LIBPNG_CONF_ENV += LIBS=-lpthread
+
 ifeq ($(BR2_ARM_CPU_HAS_NEON),y)
 LIBPNG_CONF_OPTS += --enable-arm-neon
 else
diff -Naurp buildroot-2017.05-rc2/package/libtirpc/libtirpc.mk buildroot-2017.05-rc2-patch/package/libtirpc/libtirpc.mk
--- buildroot-2017.05-rc2/package/libtirpc/libtirpc.mk	2017-05-17 10:27:16.027844496 +0200
+++ buildroot-2017.05-rc2-patch/package/libtirpc/libtirpc.mk	2017-09-15 16:08:41.000000000 +0200
@@ -12,6 +12,7 @@ LIBTIRPC_LICENSE_FILES = COPYING
 
 LIBTIRPC_INSTALL_STAGING = YES
 LIBTIRPC_AUTORECONF = YES
+LIBTIRPC_LIBTOOL_PATCH = NO
 
 # getrpcby{number,name} are only provided if 'GQ' is defined
 LIBTIRPC_CONF_ENV = CFLAGS="$(TARGET_CFLAGS) -DGQ"
diff -Naurp buildroot-2017.05-rc2/package/libxml2/libxml2.mk buildroot-2017.05-rc2-patch/package/libxml2/libxml2.mk
--- buildroot-2017.05-rc2/package/libxml2/libxml2.mk	2017-05-17 10:27:16.027844496 +0200
+++ buildroot-2017.05-rc2-patch/package/libxml2/libxml2.mk	2017-09-15 19:06:30.000000000 +0200
@@ -16,7 +16,7 @@ ifeq ($(BR2_m68k_cf),y)
 LIBXML2_CONF_ENV += CFLAGS="$(TARGET_CFLAGS) -mxgot"
 endif
 
-LIBXML2_CONF_OPTS = --with-gnu-ld --without-python --without-debug
+LIBXML2_CONF_OPTS = --with-gnu-ld --with-python --without-debug
 
 HOST_LIBXML2_DEPENDENCIES = host-pkgconf
 LIBXML2_DEPENDENCIES = host-pkgconf
diff -Naurp buildroot-2017.05-rc2/package/libxml-parser-perl/Config.in buildroot-2017.05-rc2-patch/package/libxml-parser-perl/Config.in
--- buildroot-2017.05-rc2/package/libxml-parser-perl/Config.in	1970-01-01 01:00:00.000000000 +0100
+++ buildroot-2017.05-rc2-patch/package/libxml-parser-perl/Config.in	2017-09-15 16:08:41.000000000 +0200
@@ -0,0 +1,7 @@
+config BR2_PACKAGE_LIBXML_PARSER_PERL
+	bool "libxml-parser-perl"
+	select BR2_PACKAGE_PERL
+	help
+	  libxml-parser-perl
+
+	  https://......
diff -Naurp buildroot-2017.05-rc2/package/libxml-parser-perl/libxml-parser-perl.mk buildroot-2017.05-rc2-patch/package/libxml-parser-perl/libxml-parser-perl.mk
--- buildroot-2017.05-rc2/package/libxml-parser-perl/libxml-parser-perl.mk	2017-05-17 10:27:16.027844496 +0200
+++ buildroot-2017.05-rc2-patch/package/libxml-parser-perl/libxml-parser-perl.mk	2017-09-15 16:08:41.000000000 +0200
@@ -7,28 +7,32 @@
 LIBXML_PARSER_PERL_VERSION = 2.44
 LIBXML_PARSER_PERL_SOURCE = XML-Parser-$(LIBXML_PARSER_PERL_VERSION).tar.gz
 LIBXML_PARSER_PERL_SITE = $(BR2_CPAN_MIRROR)/authors/id/T/TO/TODDR
-HOST_LIBXML_PARSER_PERL_DEPENDENCIES = host-expat
+LIBXML_PARSER_PERL_DEPENDENCIES = expat
 LIBXML_PARSER_PERL_LICENSE = Artistic or GPL-1.0+
 LIBXML_PARSER_PERL_RUN_PERL = `which perl`
+LIBXML_PARSER_PERL_INSTALL_TARGET = YES
 
-define HOST_LIBXML_PARSER_PERL_CONFIGURE_CMDS
+LIBXML_PARSER_PERL_DEPENDENCIES += perl
+
+define LIBXML_PARSER_PERL_CONFIGURE_CMDS
 	(cd $(@D) ; \
-		$(HOST_CONFIGURE_OPTS) $(LIBXML_PARSER_PERL_RUN_PERL) Makefile.PL \
+		$(CONFIGURE_OPTS) $(LIBXML_PARSER_PERL_RUN_PERL) Makefile.PL \
 			PREFIX=$(HOST_DIR)/usr \
 			EXPATLIBPATH=$(HOST_DIR)/usr/lib \
 			EXPATINCPATH=$(HOST_DIR)/usr/include \
-			INSTALLDIRS=site \
-			INSTALLSITELIB=$(HOST_DIR)/usr/lib/perl \
-			INSTALLSITEARCH=$(HOST_DIR)/usr/lib/perl \
-	)
+			INSTALLDIRS=site)
+endef
+
+define LIBXML_PARSER_PERL_BUILD_CMDS
+	$(MAKE_ENV) $(MAKE) -C $(@D)
 endef
 
-define HOST_LIBXML_PARSER_PERL_BUILD_CMDS
-	$(HOST_MAKE_ENV) $(MAKE) -C $(@D)
+define LIBXML_PARSER_PERL_INSTALL_CMDS
+	$(MAKE_ENV) $(MAKE) -C $(@D) install
 endef
 
-define HOST_LIBXML_PARSER_PERL_INSTALL_CMDS
-	$(HOST_MAKE_ENV) $(MAKE) -C $(@D) install
+define LIBXML_PARSER_PERL_INSTALL_TARGET_CMDS
+	$(TARGET_MAKE_ENV) $(MAKE) DESTDIR=$(TARGET_DIR) -C $(@D) install
 endef
 
-$(eval $(host-generic-package))
+$(eval $(generic-package))
diff -Naurp buildroot-2017.05-rc2/package/libxslt/libxslt.mk buildroot-2017.05-rc2-patch/package/libxslt/libxslt.mk
--- buildroot-2017.05-rc2/package/libxslt/libxslt.mk	2017-05-17 10:27:16.027844496 +0200
+++ buildroot-2017.05-rc2-patch/package/libxslt/libxslt.mk	2017-09-15 16:08:41.000000000 +0200
@@ -13,11 +13,11 @@ LIBXSLT_LICENSE_FILES = COPYING
 LIBXSLT_CONF_OPTS = \
 	--with-gnu-ld \
 	--without-debug \
-	--without-python \
+	--with-python \
 	--with-libxml-prefix=$(STAGING_DIR)/usr/ \
 	--with-libxml-libs-prefix=$(STAGING_DIR)/usr/lib
 LIBXSLT_CONFIG_SCRIPTS = xslt-config
-LIBXSLT_DEPENDENCIES = libxml2
+LIBXSLT_DEPENDENCIES = libxml2 python
 
 # GCC bug with Os/O2/O3, PR77311
 # error: unable to find a register to spill in class 'CCREGS'
diff -Naurp buildroot-2017.05-rc2/package/linux-headers/linux-headers.mk buildroot-2017.05-rc2-patch/package/linux-headers/linux-headers.mk
--- buildroot-2017.05-rc2/package/linux-headers/linux-headers.mk	2017-05-17 10:27:16.027844496 +0200
+++ buildroot-2017.05-rc2-patch/package/linux-headers/linux-headers.mk	2017-09-15 16:08:41.000000000 +0200
@@ -122,13 +122,30 @@ define LINUX_HEADERS_INSTALL_STAGING_CMD
 			headers_install)
 endef
 
+define LINUX_HEADERS_INSTALL_TARGET_CMDS
+	(cd $(@D); \
+		$(TARGET_MAKE_ENV) $(MAKE) \
+			ARCH=$(KERNEL_ARCH) \
+			HOSTCC="$(HOSTCC)" \
+			HOSTCFLAGS="$(HOSTCFLAGS)" \
+			HOSTCXX="$(HOSTCXX)" \
+			INSTALL_HDR_PATH=$(TARGET_DIR)/usr \
+			headers_install)
+endef
+
 ifeq ($(BR2_KERNEL_HEADERS_VERSION)$(BR2_KERNEL_HEADERS_AS_KERNEL),y)
-define LINUX_HEADERS_CHECK_VERSION
+define LINUX_HEADERS_CHECK_VERSION_STAGING
 	$(call check_kernel_headers_version,\
 		$(STAGING_DIR),\
 		$(call qstrip,$(BR2_TOOLCHAIN_HEADERS_AT_LEAST)))
 endef
-LINUX_HEADERS_POST_INSTALL_STAGING_HOOKS += LINUX_HEADERS_CHECK_VERSION
+define LINUX_HEADERS_CHECK_VERSION_TARGET
+	$(call check_kernel_headers_version,\
+		$(TARGET_DIR),\
+		$(call qstrip,$(BR2_TOOLCHAIN_HEADERS_AT_LEAST)))
+endef
+LINUX_HEADERS_POST_INSTALL_STAGING_HOOKS += LINUX_HEADERS_CHECK_VERSION_STAGING
+LINUX_HEADERS_POST_INSTALL_TARGET_HOOKS += LINUX_HEADERS_CHECK_VERSION_TARGET
 endif
 
 $(eval $(generic-package))
diff -Naurp buildroot-2017.05-rc2/package/lirc-tools/Config.in buildroot-2017.05-rc2-patch/package/lirc-tools/Config.in
--- buildroot-2017.05-rc2/package/lirc-tools/Config.in	2017-05-17 10:27:16.027844496 +0200
+++ buildroot-2017.05-rc2-patch/package/lirc-tools/Config.in	2017-09-15 16:08:41.000000000 +0200
@@ -5,6 +5,7 @@ config BR2_PACKAGE_LIRC_TOOLS
 	depends on BR2_INSTALL_LIBSTDCPP
 	# dlopen()
 	depends on !BR2_STATIC_LIBS
+	select BR2_PACKAGE_PYTHON3
 	help
 	  LIRC is a package that supports receiving and sending IR
 	  signals of the most common IR remote controls. It contains a
diff -Naurp buildroot-2017.05-rc2/package/lm-sensors/lm-sensors.mk buildroot-2017.05-rc2-patch/package/lm-sensors/lm-sensors.mk
--- buildroot-2017.05-rc2/package/lm-sensors/lm-sensors.mk	2017-05-17 10:27:16.027844496 +0200
+++ buildroot-2017.05-rc2-patch/package/lm-sensors/lm-sensors.mk	2017-09-12 11:30:16.000000000 +0200
@@ -43,12 +43,10 @@ endef
 
 define LM_SENSORS_INSTALL_STAGING_CMDS
 	$(TARGET_MAKE_ENV) $(MAKE) -C $(@D) $(LM_SENSORS_MAKE_OPTS) DESTDIR=$(STAGING_DIR) install
-	rm -f $(addprefix $(STAGING_DIR)/usr/,$(LM_SENSORS_BINS_) $(LM_SENSORS_BINS_y))
 endef
 
 define LM_SENSORS_INSTALL_TARGET_CMDS
 	$(TARGET_MAKE_ENV) $(MAKE) -C $(@D) $(LM_SENSORS_MAKE_OPTS) DESTDIR=$(TARGET_DIR) install
-	rm -f $(addprefix $(TARGET_DIR)/usr/,$(LM_SENSORS_BINS_))
 endef
 
 $(eval $(generic-package))
diff -Naurp buildroot-2017.05-rc2/package/lzo/lzo.mk buildroot-2017.05-rc2-patch/package/lzo/lzo.mk
--- buildroot-2017.05-rc2/package/lzo/lzo.mk	2017-05-17 10:27:16.027844496 +0200
+++ buildroot-2017.05-rc2-patch/package/lzo/lzo.mk	2017-09-15 16:08:41.000000000 +0200
@@ -12,6 +12,7 @@ LZO_INSTALL_STAGING = YES
 # Ships a beta libtool version hence our patch doesn't apply.
 # Run autoreconf to regenerate ltmain.sh.
 LZO_AUTORECONF = YES
+LZO_LIBTOOL_PATCH = NO
 
 $(eval $(autotools-package))
 $(eval $(host-autotools-package))
diff -Naurp buildroot-2017.05-rc2/package/m4/Config.in buildroot-2017.05-rc2-patch/package/m4/Config.in
--- buildroot-2017.05-rc2/package/m4/Config.in	1970-01-01 01:00:00.000000000 +0100
+++ buildroot-2017.05-rc2-patch/package/m4/Config.in	2017-09-15 16:08:41.000000000 +0200
@@ -0,0 +1,10 @@
+config BR2_PACKAGE_M4
+	bool "m4"
+	depends on BR2_TOOLCHAIN_HAS_THREADS
+	help
+	  m4
+
+	  https://......
+
+comment "iw needs a toolchain w/ threads"
+	depends on !BR2_TOOLCHAIN_HAS_THREADS
diff -Naurp buildroot-2017.05-rc2/package/m4/m4.mk buildroot-2017.05-rc2-patch/package/m4/m4.mk
--- buildroot-2017.05-rc2/package/m4/m4.mk	2017-05-17 10:27:16.027844496 +0200
+++ buildroot-2017.05-rc2-patch/package/m4/m4.mk	2017-09-15 16:08:41.000000000 +0200
@@ -11,4 +11,5 @@ M4_LICENSE = GPL-3.0+
 M4_LICENSE_FILES = COPYING
 HOST_M4_CONF_OPTS = --disable-static
 
+$(eval $(autotools-package))
 $(eval $(host-autotools-package))
diff -Naurp buildroot-2017.05-rc2/package/mesa3d-headers/mesa3d-headers.mk buildroot-2017.05-rc2-patch/package/mesa3d-headers/mesa3d-headers.mk
--- buildroot-2017.05-rc2/package/mesa3d-headers/mesa3d-headers.mk	2017-05-17 10:27:16.027844496 +0200
+++ buildroot-2017.05-rc2-patch/package/mesa3d-headers/mesa3d-headers.mk	2017-09-15 16:08:41.000000000 +0200
@@ -20,7 +20,7 @@ MESA3D_HEADERS_LICENSE_FILES = docs/lice
 
 # Only installs header files
 MESA3D_HEADERS_INSTALL_STAGING = YES
-MESA3D_HEADERS_INSTALL_TARGET = NO
+MESA3D_HEADERS_INSTALL_TARGET = YES
 
 MESA3D_HEADERS_DIRS = KHR
 
diff -Naurp buildroot-2017.05-rc2/package/musl/musl.mk buildroot-2017.05-rc2-patch/package/musl/musl.mk
--- buildroot-2017.05-rc2/package/musl/musl.mk	2017-05-17 10:27:16.027844496 +0200
+++ buildroot-2017.05-rc2-patch/package/musl/musl.mk	2017-09-15 16:08:41.000000000 +0200
@@ -57,8 +57,7 @@ endef
 
 define MUSL_INSTALL_TARGET_CMDS
 	$(TARGET_MAKE_ENV) $(MAKE) -C $(@D) \
-		DESTDIR=$(TARGET_DIR) install-libs
-	$(RM) $(addprefix $(TARGET_DIR)/lib/,crt1.o crtn.o crti.o Scrt1.o)
+		DESTDIR=$(TARGET_DIR) install-libs install-tools install-headers
 endef
 
 $(eval $(generic-package))
diff -Naurp buildroot-2017.05-rc2/package/musl-compat-headers/musl-compat-headers.mk buildroot-2017.05-rc2-patch/package/musl-compat-headers/musl-compat-headers.mk
--- buildroot-2017.05-rc2/package/musl-compat-headers/musl-compat-headers.mk	2017-05-17 10:27:16.027844496 +0200
+++ buildroot-2017.05-rc2-patch/package/musl-compat-headers/musl-compat-headers.mk	2017-09-15 16:08:41.000000000 +0200
@@ -31,4 +31,11 @@ define MUSL_COMPAT_HEADERS_INSTALL_STAGI
 		$(STAGING_DIR)/usr/include/sys/cdefs.h
 endef
 
+define MUSL_COMPAT_HEADERS_INSTALL_TARGET_CMDS
+	$(INSTALL) -D -m 0644 $(@D)/queue.h \
+		$(TARGET_DIR)/usr/include/sys/queue.h
+	$(INSTALL) -D -m 0644 $(@D)/cdefs.h \
+		$(TARGET_DIR)/usr/include/sys/cdefs.h
+endef
+
 $(eval $(generic-package))
diff -Naurp buildroot-2017.05-rc2/package/nasm/Config.in buildroot-2017.05-rc2-patch/package/nasm/Config.in
--- buildroot-2017.05-rc2/package/nasm/Config.in	1970-01-01 01:00:00.000000000 +0100
+++ buildroot-2017.05-rc2-patch/package/nasm/Config.in	2017-09-15 16:08:41.000000000 +0200
@@ -0,0 +1,6 @@
+config BR2_PACKAGE_NASM
+	bool "nasm"
+	help
+	  nasm
+
+	  http://....
diff -Naurp buildroot-2017.05-rc2/package/nasm/nasm.mk buildroot-2017.05-rc2-patch/package/nasm/nasm.mk
--- buildroot-2017.05-rc2/package/nasm/nasm.mk	2017-05-17 10:27:16.027844496 +0200
+++ buildroot-2017.05-rc2-patch/package/nasm/nasm.mk	2017-09-15 16:08:41.000000000 +0200
@@ -10,4 +10,9 @@ NASM_SITE = http://www.nasm.us/pub/nasm/
 NASM_LICENSE = BSD-2-Clause
 NASM_LICENSE_FILES = LICENSE
 
+define NASM_INSTALL_TARGET_CMDS
+	$(TARGET_MAKE_ENV) $(MAKE) -C $(@D) INSTALLROOT=$(TARGET_DIR) install
+endef
+
+$(eval $(autotools-package))
 $(eval $(host-autotools-package))
diff -Naurp buildroot-2017.05-rc2/package/nfs-utils/nfs-utils.mk buildroot-2017.05-rc2-patch/package/nfs-utils/nfs-utils.mk
--- buildroot-2017.05-rc2/package/nfs-utils/nfs-utils.mk	2017-05-17 10:27:16.027844496 +0200
+++ buildroot-2017.05-rc2-patch/package/nfs-utils/nfs-utils.mk	2017-09-15 16:08:41.000000000 +0200
@@ -10,6 +10,7 @@ NFS_UTILS_SITE = https://www.kernel.org/
 NFS_UTILS_LICENSE = GPL-2.0+
 NFS_UTILS_LICENSE_FILES = COPYING
 NFS_UTILS_AUTORECONF = YES
+NFS_UTILS_LIBTOOL_PATCH = NO
 NFS_UTILS_DEPENDENCIES = host-pkgconf
 
 NFS_UTILS_CONF_ENV = knfsd_cv_bsd_signals=no
diff -Naurp buildroot-2017.05-rc2/package/parted/parted.mk buildroot-2017.05-rc2-patch/package/parted/parted.mk
--- buildroot-2017.05-rc2/package/parted/parted.mk	2017-05-17 10:27:16.027844496 +0200
+++ buildroot-2017.05-rc2-patch/package/parted/parted.mk	2017-09-15 16:08:41.000000000 +0200
@@ -10,8 +10,8 @@ PARTED_SITE = $(BR2_GNU_MIRROR)/parted
 PARTED_DEPENDENCIES = host-pkgconf util-linux
 PARTED_INSTALL_STAGING = YES
 # For uclinux patch
-PARTED_AUTORECONF = YES
-PARTED_GETTEXTIZE = YES
+PARTED_AUTORECONF = NO
+PARTED_GETTEXTIZE = NO
 PARTED_LICENSE = GPL-3.0+
 PARTED_LICENSE_FILES = COPYING
 
diff -Naurp buildroot-2017.05-rc2/package/perl/perl.mk buildroot-2017.05-rc2-patch/package/perl/perl.mk
--- buildroot-2017.05-rc2/package/perl/perl.mk	2017-05-17 10:27:16.027844496 +0200
+++ buildroot-2017.05-rc2-patch/package/perl/perl.mk	2017-09-15 16:08:41.000000000 +0200
@@ -66,11 +66,6 @@ ifeq ($(BR2_STATIC_LIBS),y)
 PERL_CONF_OPTS += --all-static --no-dynaloader
 endif
 
-PERL_MODULES = $(call qstrip,$(BR2_PACKAGE_PERL_MODULES))
-ifneq ($(PERL_MODULES),)
-PERL_CONF_OPTS += --only-mod=$(subst $(space),$(comma),$(PERL_MODULES))
-endif
-
 define PERL_CONFIGURE_CMDS
 	(cd $(@D); $(TARGET_MAKE_ENV) HOSTCC='$(HOSTCC_NOCCACHE)' \
 		./configure $(PERL_CONF_OPTS))
diff -Naurp buildroot-2017.05-rc2/package/popt/popt.mk buildroot-2017.05-rc2-patch/package/popt/popt.mk
--- buildroot-2017.05-rc2/package/popt/popt.mk	2017-05-17 10:27:16.027844496 +0200
+++ buildroot-2017.05-rc2-patch/package/popt/popt.mk	2017-09-15 16:08:41.000000000 +0200
@@ -11,6 +11,7 @@ POPT_LICENSE = MIT
 POPT_LICENSE_FILES = COPYING
 POPT_AUTORECONF = YES
 POPT_GETTEXTIZE = YES
+POPT_LIBTOOL_PATCH = NO
 
 ifeq ($(BR2_PACKAGE_GETTEXT),y)
 POPT_DEPENDENCIES += gettext
diff -Naurp buildroot-2017.05-rc2/package/pulseaudio/pulseaudio.mk buildroot-2017.05-rc2-patch/package/pulseaudio/pulseaudio.mk
--- buildroot-2017.05-rc2/package/pulseaudio/pulseaudio.mk	2017-05-17 10:27:16.027844496 +0200
+++ buildroot-2017.05-rc2-patch/package/pulseaudio/pulseaudio.mk	2017-09-15 16:08:41.000000000 +0200
@@ -23,6 +23,7 @@ PULSEAUDIO_CONF_ENV += \
 
 # 0002-webrtc-C-11-is-only-required-for-WebRTC-support.patch
 PULSEAUDIO_AUTORECONF = YES
+PULSEAUDIO_LIBTOOL_PATCH = NO
 
 PULSEAUDIO_DEPENDENCIES = \
 	host-pkgconf libtool json-c libsndfile speex host-intltool \
diff -Naurp buildroot-2017.05-rc2/package/python3/python3.mk buildroot-2017.05-rc2-patch/package/python3/python3.mk
--- buildroot-2017.05-rc2/package/python3/python3.mk	2017-05-17 10:27:16.027844496 +0200
+++ buildroot-2017.05-rc2-patch/package/python3/python3.mk	2017-09-15 19:21:42.000000000 +0200
@@ -174,10 +174,6 @@ define HOST_PYTHON3_INSTALL_TOOLS
 endef
 HOST_PYTHON3_POST_INSTALL_HOOKS += HOST_PYTHON3_INSTALL_TOOLS
 
-PYTHON3_CONF_ENV += \
-	PGEN_FOR_BUILD=$(HOST_DIR)/usr/bin/python-pgen \
-	FREEZE_IMPORTLIB_FOR_BUILD=$(HOST_DIR)/usr/bin/python-freeze-importlib
-
 #
 # Remove useless files. In the config/ directory, only the Makefile
 # and the pyconfig.h files are needed at runtime.
diff -Naurp buildroot-2017.05-rc2/package/readline/readline.mk buildroot-2017.05-rc2-patch/package/readline/readline.mk
--- buildroot-2017.05-rc2/package/readline/readline.mk	2017-05-17 10:27:16.027844496 +0200
+++ buildroot-2017.05-rc2-patch/package/readline/readline.mk	2017-09-15 16:08:41.000000000 +0200
@@ -24,6 +24,11 @@ define READLINE_INSTALL_INPUTRC
 endef
 READLINE_POST_INSTALL_TARGET_HOOKS += READLINE_INSTALL_INPUTRC
 
+define READLINE_INSTALL_PKGCONF_DATA
+	$(INSTALL) -D -m 644 $(@D)/readline.pc $(TARGET_DIR)/usr/lib/pkgconfig/readline.pc
+endef
+READLINE_POST_INSTALL_TARGET_HOOKS += READLINE_INSTALL_PKGCONF_DATA
+
 ifneq ($(BR2_STATIC_LIBS),y)
 # libraries get installed read only, so strip fails
 define READLINE_INSTALL_FIXUPS_SHARED
diff -Naurp buildroot-2017.05-rc2/package/samba4/Config.in buildroot-2017.05-rc2-patch/package/samba4/Config.in
--- buildroot-2017.05-rc2/package/samba4/Config.in	2017-05-17 10:27:16.027844496 +0200
+++ buildroot-2017.05-rc2-patch/package/samba4/Config.in	2017-09-15 16:08:41.000000000 +0200
@@ -17,6 +17,7 @@ config BR2_PACKAGE_SAMBA4
 	select BR2_PACKAGE_POPT
 	select BR2_PACKAGE_PYTHON
 	select BR2_PACKAGE_ZLIB
+	select BR2_PACKAGE_HEIMDAL
 	help
 	  Provides secure, stable and fast file and print services
 	  for all clients using the SMB/CIFS protocol, such as all
diff -Naurp buildroot-2017.05-rc2/package/samba4/samba-4.5.10-lib64.patch buildroot-2017.05-rc2-patch/package/samba4/samba-4.5.10-lib64.patch
--- buildroot-2017.05-rc2/package/samba4/samba-4.5.10-lib64.patch	1970-01-01 01:00:00.000000000 +0100
+++ buildroot-2017.05-rc2-patch/package/samba4/samba-4.5.10-lib64.patch	2017-09-15 16:08:41.000000000 +0200
@@ -0,0 +1,48 @@
+diff -Naurp samba-4.5.10/buildtools/wafsamba/samba_conftests.py samba-4.5.10-lib64/buildtools/wafsamba/samba_conftests.py
+--- samba-4.5.10/buildtools/wafsamba/samba_conftests.py	2016-08-11 07:51:04.000000000 +0000
++++ samba-4.5.10-lib64/buildtools/wafsamba/samba_conftests.py	2017-06-18 13:08:14.723961837 +0000
+@@ -506,7 +506,7 @@ def CHECK_STANDARD_LIBPATH(conf):
+         out = Utils.cmd_output(cmd).split('\n')
+     except ValueError:
+         # option not supported by compiler - use a standard list of directories
+-        dirlist = [ '/usr/lib', '/usr/lib64' ]
++        dirlist = [ '/usr/lib', '/usr/lib' ]
+     except:
+         raise Utils.WafError('Unexpected error running "%s"' % (cmd))
+     else:
+diff -Naurp samba-4.5.10/ctdb/configure.rpm samba-4.5.10-lib64/ctdb/configure.rpm
+--- samba-4.5.10/ctdb/configure.rpm	2016-08-11 07:51:04.000000000 +0000
++++ samba-4.5.10-lib64/ctdb/configure.rpm	2017-06-18 13:08:04.091961946 +0000
+@@ -1,7 +1,7 @@
+ #!/bin/sh
+ 
+ if gcc -dM -E - </dev/null | grep -Eq '__(x86_64|powerpc64)__' ; then
+-	_libdir=/usr/lib64
++	_libdir=/usr/lib
+ else
+ 	_libdir=/usr/lib
+ fi
+diff -Naurp samba-4.5.10/packaging/RHEL-CTDB/configure.rpm samba-4.5.10-lib64/packaging/RHEL-CTDB/configure.rpm
+--- samba-4.5.10/packaging/RHEL-CTDB/configure.rpm	2016-08-11 07:51:04.000000000 +0000
++++ samba-4.5.10-lib64/packaging/RHEL-CTDB/configure.rpm	2017-06-18 13:07:58.155962006 +0000
+@@ -2,7 +2,7 @@
+ 
+ case `uname -m` in
+ 	x86_64)
+-		_libarch="/lib64"
++		_libarch="/lib"
+ 		;;
+ 	*)
+ 		_libarch="/lib"
+diff -Naurp samba-4.5.10/source3/modules/lib_vxfs.c samba-4.5.10-lib64/source3/modules/lib_vxfs.c
+--- samba-4.5.10/source3/modules/lib_vxfs.c	2016-08-11 07:51:04.000000000 +0000
++++ samba-4.5.10-lib64/source3/modules/lib_vxfs.c	2017-06-18 13:08:19.735961786 +0000
+@@ -27,7 +27,7 @@
+  * Available under GPL at
+  * http://www.veritas.com/community/downloads/vxfsmisc-library
+  */
+-#define LIBVXFS "/usr/lib64/vxfsmisc.so"
++#define LIBVXFS "/usr/lib/vxfsmisc.so"
+ 
+ 
+ static int (*vxfs_setxattr_fd_func) (int fd, const char *name,
diff -Naurp buildroot-2017.05-rc2/package/screen/screen.mk buildroot-2017.05-rc2-patch/package/screen/screen.mk
--- buildroot-2017.05-rc2/package/screen/screen.mk	2017-05-17 10:27:16.027844496 +0200
+++ buildroot-2017.05-rc2-patch/package/screen/screen.mk	2017-09-15 16:08:41.000000000 +0200
@@ -11,7 +11,7 @@ SCREEN_LICENSE_FILES = COPYING
 SCREEN_DEPENDENCIES = ncurses
 SCREEN_AUTORECONF = YES
 SCREEN_CONF_ENV = CFLAGS="$(TARGET_CFLAGS)"
-SCREEN_CONF_OPTS = --enable-colors256
+SCREEN_CONF_OPTS = --enable-colors256 --with-sys-screenrc=/etc/screenrc
 SCREEN_INSTALL_TARGET_OPTS = DESTDIR=$(TARGET_DIR) SCREEN=screen install_bin
 
 define SCREEN_INSTALL_SCREENRC
diff -Naurp buildroot-2017.05-rc2/package/systemd/systemd.mk buildroot-2017.05-rc2-patch/package/systemd/systemd.mk
--- buildroot-2017.05-rc2/package/systemd/systemd.mk	2017-05-17 10:27:16.027844496 +0200
+++ buildroot-2017.05-rc2-patch/package/systemd/systemd.mk	2017-09-15 16:08:41.000000000 +0200
@@ -45,7 +45,7 @@ SYSTEMD_CFLAGS = $(TARGET_CFLAGS) -fno-l
 # Override path to kmod, used in kmod-static-nodes.service
 SYSTEMD_CONF_ENV = \
 	CFLAGS="$(SYSTEMD_CFLAGS)" \
-	ac_cv_path_KMOD=/usr/bin/kmod
+	ac_cv_path_KMOD=/bin/kmod
 
 define SYSTEMD_RUN_INTLTOOLIZE
 	cd $(@D) && $(HOST_DIR)/usr/bin/intltoolize --force --automake
diff -Naurp buildroot-2017.05-rc2/package/tar/tar.mk buildroot-2017.05-rc2-patch/package/tar/tar.mk
--- buildroot-2017.05-rc2/package/tar/tar.mk	2017-05-17 10:27:16.027844496 +0200
+++ buildroot-2017.05-rc2-patch/package/tar/tar.mk	2017-09-15 16:08:41.000000000 +0200
@@ -9,7 +9,8 @@ TAR_SOURCE = tar-$(TAR_VERSION).tar.xz
 TAR_SITE = $(BR2_GNU_MIRROR)/tar
 # busybox installs in /bin, so we need tar to install as well in /bin
 # so that it overrides the Busybox symlinks.
-TAR_CONF_OPTS = --exec-prefix=/
+TAR_CONF_OPTS = --exec-prefix=/ --libexecdir=/usr/libexec
+TAR_CONF_ENV = FORCE_UNSAFE_CONFIGURE=1
 TAR_LICENSE = GPL-3.0+
 TAR_LICENSE_FILES = COPYING
 
diff -Naurp buildroot-2017.05-rc2/package/texinfo/Config.in buildroot-2017.05-rc2-patch/package/texinfo/Config.in
--- buildroot-2017.05-rc2/package/texinfo/Config.in	1970-01-01 01:00:00.000000000 +0100
+++ buildroot-2017.05-rc2-patch/package/texinfo/Config.in	2017-05-28 21:13:39.000000000 +0200
@@ -0,0 +1,6 @@
+config BR2_PACKAGE_TEXINFO
+	bool "texinfo"
+	help
+	  texinfo
+
+	  https://......
diff -Naurp buildroot-2017.05-rc2/package/texinfo/texinfo.hash buildroot-2017.05-rc2-patch/package/texinfo/texinfo.hash
--- buildroot-2017.05-rc2/package/texinfo/texinfo.hash	1970-01-01 01:00:00.000000000 +0100
+++ buildroot-2017.05-rc2-patch/package/texinfo/texinfo.hash	2017-05-28 21:22:35.000000000 +0200
@@ -0,0 +1,2 @@
+# Locally calculated after checking pgp signature
+sha256	246cf3ffa54985118ec2eea2b8d0c71b92114efe6282c2ae90d65029db4cf93a	texinfo-6.3.tar.xz
diff -Naurp buildroot-2017.05-rc2/package/texinfo/texinfo.mk buildroot-2017.05-rc2-patch/package/texinfo/texinfo.mk
--- buildroot-2017.05-rc2/package/texinfo/texinfo.mk	1970-01-01 01:00:00.000000000 +0100
+++ buildroot-2017.05-rc2-patch/package/texinfo/texinfo.mk	2017-05-28 22:46:03.000000000 +0200
@@ -0,0 +1,13 @@
+################################################################################
+#
+# texinfo
+#
+################################################################################
+
+TEXINFO_VERSION = 6.3
+TEXINFO_SOURCE = texinfo-$(TEXINFO_VERSION).tar.xz
+TEXINFO_SITE = https://ftp.osuosl.org/pub/lfs/lfs-packages/8.0
+TEXINFO_LICENSE = GPL-3.0+
+TEXINFO_LICENSE_FILES = COPYING
+
+$(eval $(autotools-package))
diff -Naurp buildroot-2017.05-rc2/package/util-linux/util-linux.mk buildroot-2017.05-rc2-patch/package/util-linux/util-linux.mk
--- buildroot-2017.05-rc2/package/util-linux/util-linux.mk	2017-05-17 10:27:16.027844496 +0200
+++ buildroot-2017.05-rc2-patch/package/util-linux/util-linux.mk	2017-09-15 19:57:59.000000000 +0200
@@ -32,6 +32,8 @@ UTIL_LINUX_CONF_OPTS += \
 	--without-systemd \
 	--with-systemdsystemunitdir=no
 
+UTIL_LINUX_CONF_OPTS += --enable-all-programs
+
 # We don't want the host-busybox dependency to be added automatically
 HOST_UTIL_LINUX_DEPENDENCIES = host-pkgconf
 
diff -Naurp buildroot-2017.05-rc2/package/wayland-protocols/wayland-protocols.mk buildroot-2017.05-rc2-patch/package/wayland-protocols/wayland-protocols.mk
--- buildroot-2017.05-rc2/package/wayland-protocols/wayland-protocols.mk	2017-05-17 10:27:16.027844496 +0200
+++ buildroot-2017.05-rc2-patch/package/wayland-protocols/wayland-protocols.mk	2017-05-27 07:39:31.000000000 +0200
@@ -10,6 +10,6 @@ WAYLAND_PROTOCOLS_SOURCE = wayland-proto
 WAYLAND_PROTOCOLS_LICENSE = MIT
 WAYLAND_PROTOCOLS_LICENSE_FILES = COPYING
 WAYLAND_PROTOCOLS_INSTALL_STAGING = YES
-WAYLAND_PROTOCOLS_INSTALL_TARGET = NO
+WAYLAND_PROTOCOLS_INSTALL_TARGET = YES
 
 $(eval $(autotools-package))
diff -Naurp buildroot-2017.05-rc2/package/x11r7/xfont_font-util/xfont_font-util.mk buildroot-2017.05-rc2-patch/package/x11r7/xfont_font-util/xfont_font-util.mk
--- buildroot-2017.05-rc2/package/x11r7/xfont_font-util/xfont_font-util.mk	2017-05-17 10:27:16.027844496 +0200
+++ buildroot-2017.05-rc2-patch/package/x11r7/xfont_font-util/xfont_font-util.mk	2017-05-27 07:30:55.000000000 +0200
@@ -13,7 +13,7 @@ XFONT_FONT_UTIL_LICENSE_FILES = COPYING
 XFONT_FONT_UTIL_DEPENDENCIES = host-pkgconf
 HOST_XFONT_FONT_UTIL_DEPENDENCIES = host-pkgconf
 XFONT_FONT_UTIL_INSTALL_STAGING = YES
-XFONT_FONT_UTIL_INSTALL_TARGET = NO
+XFONT_FONT_UTIL_INSTALL_TARGET = YES
 
 $(eval $(autotools-package))
 $(eval $(host-autotools-package))
diff -Naurp buildroot-2017.05-rc2/package/x11r7/xproto_applewmproto/xproto_applewmproto.mk buildroot-2017.05-rc2-patch/package/x11r7/xproto_applewmproto/xproto_applewmproto.mk
--- buildroot-2017.05-rc2/package/x11r7/xproto_applewmproto/xproto_applewmproto.mk	2017-05-17 10:27:16.027844496 +0200
+++ buildroot-2017.05-rc2-patch/package/x11r7/xproto_applewmproto/xproto_applewmproto.mk	2017-05-27 07:30:55.000000000 +0200
@@ -10,6 +10,6 @@ XPROTO_APPLEWMPROTO_SITE = http://xorg.f
 XPROTO_APPLEWMPROTO_LICENSE = MIT
 XPROTO_APPLEWMPROTO_LICENSE_FILES = COPYING
 XPROTO_APPLEWMPROTO_INSTALL_STAGING = YES
-XPROTO_APPLEWMPROTO_INSTALL_TARGET = NO
+XPROTO_APPLEWMPROTO_INSTALL_TARGET = YES
 
 $(eval $(autotools-package))
diff -Naurp buildroot-2017.05-rc2/package/x11r7/xproto_bigreqsproto/xproto_bigreqsproto.mk buildroot-2017.05-rc2-patch/package/x11r7/xproto_bigreqsproto/xproto_bigreqsproto.mk
--- buildroot-2017.05-rc2/package/x11r7/xproto_bigreqsproto/xproto_bigreqsproto.mk	2017-05-17 10:27:16.027844496 +0200
+++ buildroot-2017.05-rc2-patch/package/x11r7/xproto_bigreqsproto/xproto_bigreqsproto.mk	2017-05-27 07:30:55.000000000 +0200
@@ -10,6 +10,6 @@ XPROTO_BIGREQSPROTO_SITE = http://xorg.f
 XPROTO_BIGREQSPROTO_LICENSE = MIT
 XPROTO_BIGREQSPROTO_LICENSE_FILES = COPYING
 XPROTO_BIGREQSPROTO_INSTALL_STAGING = YES
-XPROTO_BIGREQSPROTO_INSTALL_TARGET = NO
+XPROTO_BIGREQSPROTO_INSTALL_TARGET = YES
 
 $(eval $(autotools-package))
diff -Naurp buildroot-2017.05-rc2/package/x11r7/xproto_compositeproto/xproto_compositeproto.mk buildroot-2017.05-rc2-patch/package/x11r7/xproto_compositeproto/xproto_compositeproto.mk
--- buildroot-2017.05-rc2/package/x11r7/xproto_compositeproto/xproto_compositeproto.mk	2017-05-17 10:27:16.027844496 +0200
+++ buildroot-2017.05-rc2-patch/package/x11r7/xproto_compositeproto/xproto_compositeproto.mk	2017-05-27 07:30:55.000000000 +0200
@@ -10,6 +10,6 @@ XPROTO_COMPOSITEPROTO_SITE = http://xorg
 XPROTO_COMPOSITEPROTO_LICENSE = MIT
 XPROTO_COMPOSITEPROTO_LICENSE_FILES = COPYING
 XPROTO_COMPOSITEPROTO_INSTALL_STAGING = YES
-XPROTO_COMPOSITEPROTO_INSTALL_TARGET = NO
+XPROTO_COMPOSITEPROTO_INSTALL_TARGET = YES
 
 $(eval $(autotools-package))
diff -Naurp buildroot-2017.05-rc2/package/x11r7/xproto_damageproto/xproto_damageproto.mk buildroot-2017.05-rc2-patch/package/x11r7/xproto_damageproto/xproto_damageproto.mk
--- buildroot-2017.05-rc2/package/x11r7/xproto_damageproto/xproto_damageproto.mk	2017-05-17 10:27:16.027844496 +0200
+++ buildroot-2017.05-rc2-patch/package/x11r7/xproto_damageproto/xproto_damageproto.mk	2017-05-27 07:30:55.000000000 +0200
@@ -10,6 +10,6 @@ XPROTO_DAMAGEPROTO_SITE = http://xorg.fr
 XPROTO_DAMAGEPROTO_LICENSE = MIT
 XPROTO_DAMAGEPROTO_LICENSE_FILES = COPYING
 XPROTO_DAMAGEPROTO_INSTALL_STAGING = YES
-XPROTO_DAMAGEPROTO_INSTALL_TARGET = NO
+XPROTO_DAMAGEPROTO_INSTALL_TARGET = YES
 
 $(eval $(autotools-package))
diff -Naurp buildroot-2017.05-rc2/package/x11r7/xproto_dmxproto/xproto_dmxproto.mk buildroot-2017.05-rc2-patch/package/x11r7/xproto_dmxproto/xproto_dmxproto.mk
--- buildroot-2017.05-rc2/package/x11r7/xproto_dmxproto/xproto_dmxproto.mk	2017-05-17 10:27:16.027844496 +0200
+++ buildroot-2017.05-rc2-patch/package/x11r7/xproto_dmxproto/xproto_dmxproto.mk	2017-05-27 07:30:55.000000000 +0200
@@ -10,6 +10,6 @@ XPROTO_DMXPROTO_SITE = http://xorg.freed
 XPROTO_DMXPROTO_LICENSE = MIT
 XPROTO_DMXPROTO_LICENSE_FILES = COPYING
 XPROTO_DMXPROTO_INSTALL_STAGING = YES
-XPROTO_DMXPROTO_INSTALL_TARGET = NO
+XPROTO_DMXPROTO_INSTALL_TARGET = YES
 
 $(eval $(autotools-package))
diff -Naurp buildroot-2017.05-rc2/package/x11r7/xproto_dri2proto/xproto_dri2proto.mk buildroot-2017.05-rc2-patch/package/x11r7/xproto_dri2proto/xproto_dri2proto.mk
--- buildroot-2017.05-rc2/package/x11r7/xproto_dri2proto/xproto_dri2proto.mk	2017-05-17 10:27:16.027844496 +0200
+++ buildroot-2017.05-rc2-patch/package/x11r7/xproto_dri2proto/xproto_dri2proto.mk	2017-05-27 07:30:55.000000000 +0200
@@ -10,6 +10,6 @@ XPROTO_DRI2PROTO_SITE = http://xorg.free
 XPROTO_DRI2PROTO_LICENSE = MIT
 XPROTO_DRI2PROTO_LICENSE_FILES = COPYING
 XPROTO_DRI2PROTO_INSTALL_STAGING = YES
-XPROTO_DRI2PROTO_INSTALL_TARGET = NO
+XPROTO_DRI2PROTO_INSTALL_TARGET = YES
 
 $(eval $(autotools-package))
diff -Naurp buildroot-2017.05-rc2/package/x11r7/xproto_dri3proto/xproto_dri3proto.mk buildroot-2017.05-rc2-patch/package/x11r7/xproto_dri3proto/xproto_dri3proto.mk
--- buildroot-2017.05-rc2/package/x11r7/xproto_dri3proto/xproto_dri3proto.mk	2017-05-17 10:27:16.027844496 +0200
+++ buildroot-2017.05-rc2-patch/package/x11r7/xproto_dri3proto/xproto_dri3proto.mk	2017-05-27 07:30:55.000000000 +0200
@@ -10,6 +10,6 @@ XPROTO_DRI3PROTO_SITE = http://xorg.free
 XPROTO_DRI3PROTO_LICENSE = MIT
 XPROTO_DRI3PROTO_INSTALL_STAGING = YES
 # this package does not contain any binary files
-XPROTO_DRI3PROTO_INSTALL_TARGET = NO
+XPROTO_DRI3PROTO_INSTALL_TARGET = YES
 
 $(eval $(autotools-package))
diff -Naurp buildroot-2017.05-rc2/package/x11r7/xproto_fixesproto/xproto_fixesproto.mk buildroot-2017.05-rc2-patch/package/x11r7/xproto_fixesproto/xproto_fixesproto.mk
--- buildroot-2017.05-rc2/package/x11r7/xproto_fixesproto/xproto_fixesproto.mk	2017-05-17 10:27:16.027844496 +0200
+++ buildroot-2017.05-rc2-patch/package/x11r7/xproto_fixesproto/xproto_fixesproto.mk	2017-05-27 07:30:55.000000000 +0200
@@ -10,7 +10,7 @@ XPROTO_FIXESPROTO_SITE = http://xorg.fre
 XPROTO_FIXESPROTO_LICENSE = MIT
 XPROTO_FIXESPROTO_LICENSE_FILES = COPYING
 XPROTO_FIXESPROTO_INSTALL_STAGING = YES
-XPROTO_FIXESPROTO_INSTALL_TARGET = NO
+XPROTO_FIXESPROTO_INSTALL_TARGET = YES
 
 $(eval $(autotools-package))
 $(eval $(host-autotools-package))
diff -Naurp buildroot-2017.05-rc2/package/x11r7/xproto_fontcacheproto/xproto_fontcacheproto.mk buildroot-2017.05-rc2-patch/package/x11r7/xproto_fontcacheproto/xproto_fontcacheproto.mk
--- buildroot-2017.05-rc2/package/x11r7/xproto_fontcacheproto/xproto_fontcacheproto.mk	2017-05-17 10:27:16.027844496 +0200
+++ buildroot-2017.05-rc2-patch/package/x11r7/xproto_fontcacheproto/xproto_fontcacheproto.mk	2017-05-27 07:30:55.000000000 +0200
@@ -10,7 +10,7 @@ XPROTO_FONTCACHEPROTO_SITE = http://xorg
 XPROTO_FONTCACHEPROTO_LICENSE = MIT
 XPROTO_FONTCACHEPROTO_LICENSE_FILES = COPYING
 XPROTO_FONTCACHEPROTO_INSTALL_STAGING = YES
-XPROTO_FONTCACHEPROTO_INSTALL_TARGET = NO
+XPROTO_FONTCACHEPROTO_INSTALL_TARGET = YES
 
 $(eval $(autotools-package))
 $(eval $(host-autotools-package))
diff -Naurp buildroot-2017.05-rc2/package/x11r7/xproto_fontsproto/xproto_fontsproto.mk buildroot-2017.05-rc2-patch/package/x11r7/xproto_fontsproto/xproto_fontsproto.mk
--- buildroot-2017.05-rc2/package/x11r7/xproto_fontsproto/xproto_fontsproto.mk	2017-05-17 10:27:16.027844496 +0200
+++ buildroot-2017.05-rc2-patch/package/x11r7/xproto_fontsproto/xproto_fontsproto.mk	2017-05-27 07:30:55.000000000 +0200
@@ -10,7 +10,7 @@ XPROTO_FONTSPROTO_SITE = http://xorg.fre
 XPROTO_FONTSPROTO_LICENSE = MIT
 XPROTO_FONTSPROTO_LICENSE_FILES = COPYING
 XPROTO_FONTSPROTO_INSTALL_STAGING = YES
-XPROTO_FONTSPROTO_INSTALL_TARGET = NO
+XPROTO_FONTSPROTO_INSTALL_TARGET = YES
 
 $(eval $(autotools-package))
 $(eval $(host-autotools-package))
diff -Naurp buildroot-2017.05-rc2/package/x11r7/xproto_glproto/xproto_glproto.mk buildroot-2017.05-rc2-patch/package/x11r7/xproto_glproto/xproto_glproto.mk
--- buildroot-2017.05-rc2/package/x11r7/xproto_glproto/xproto_glproto.mk	2017-05-17 10:27:16.027844496 +0200
+++ buildroot-2017.05-rc2-patch/package/x11r7/xproto_glproto/xproto_glproto.mk	2017-05-27 07:30:55.000000000 +0200
@@ -10,6 +10,6 @@ XPROTO_GLPROTO_SITE = http://xorg.freede
 XPROTO_GLPROTO_LICENSE = MIT
 XPROTO_GLPROTO_LICENSE_FILES = COPYING
 XPROTO_GLPROTO_INSTALL_STAGING = YES
-XPROTO_GLPROTO_INSTALL_TARGET = NO
+XPROTO_GLPROTO_INSTALL_TARGET = YES
 
 $(eval $(autotools-package))
diff -Naurp buildroot-2017.05-rc2/package/x11r7/xproto_inputproto/xproto_inputproto.mk buildroot-2017.05-rc2-patch/package/x11r7/xproto_inputproto/xproto_inputproto.mk
--- buildroot-2017.05-rc2/package/x11r7/xproto_inputproto/xproto_inputproto.mk	2017-05-17 10:27:16.027844496 +0200
+++ buildroot-2017.05-rc2-patch/package/x11r7/xproto_inputproto/xproto_inputproto.mk	2017-05-27 07:30:55.000000000 +0200
@@ -10,7 +10,7 @@ XPROTO_INPUTPROTO_SITE = http://xorg.fre
 XPROTO_INPUTPROTO_LICENSE = MIT
 XPROTO_INPUTPROTO_LICENSE_FILES = COPYING
 XPROTO_INPUTPROTO_INSTALL_STAGING = YES
-XPROTO_INPUTPROTO_INSTALL_TARGET = NO
+XPROTO_INPUTPROTO_INSTALL_TARGET = YES
 
 $(eval $(autotools-package))
 $(eval $(host-autotools-package))
diff -Naurp buildroot-2017.05-rc2/package/x11r7/xproto_kbproto/xproto_kbproto.mk buildroot-2017.05-rc2-patch/package/x11r7/xproto_kbproto/xproto_kbproto.mk
--- buildroot-2017.05-rc2/package/x11r7/xproto_kbproto/xproto_kbproto.mk	2017-05-17 10:27:16.027844496 +0200
+++ buildroot-2017.05-rc2-patch/package/x11r7/xproto_kbproto/xproto_kbproto.mk	2017-05-27 07:30:55.000000000 +0200
@@ -10,7 +10,7 @@ XPROTO_KBPROTO_SITE = http://xorg.freede
 XPROTO_KBPROTO_LICENSE = MIT
 XPROTO_KBPROTO_LICENSE_FILES = COPYING
 XPROTO_KBPROTO_INSTALL_STAGING = YES
-XPROTO_KBPROTO_INSTALL_TARGET = NO
+XPROTO_KBPROTO_INSTALL_TARGET = YES
 
 $(eval $(autotools-package))
 $(eval $(host-autotools-package))
diff -Naurp buildroot-2017.05-rc2/package/x11r7/xproto_presentproto/xproto_presentproto.mk buildroot-2017.05-rc2-patch/package/x11r7/xproto_presentproto/xproto_presentproto.mk
--- buildroot-2017.05-rc2/package/x11r7/xproto_presentproto/xproto_presentproto.mk	2017-05-17 10:27:16.027844496 +0200
+++ buildroot-2017.05-rc2-patch/package/x11r7/xproto_presentproto/xproto_presentproto.mk	2017-05-27 07:30:55.000000000 +0200
@@ -11,6 +11,6 @@ XPROTO_PRESENTPROTO_LICENSE = MIT
 XPROTO_PRESENTPROTO_LICENSE_FILES = presentproto.h
 XPROTO_PRESENTPROTO_DEPENDENCIES = host-pkgconf xutil_util-macros
 XPROTO_PRESENTPROTO_INSTALL_STAGING = YES
-XPROTO_PRESENTPROTO_INSTALL_TARGET = NO
+XPROTO_PRESENTPROTO_INSTALL_TARGET = YES
 
 $(eval $(autotools-package))
diff -Naurp buildroot-2017.05-rc2/package/x11r7/xproto_randrproto/xproto_randrproto.mk buildroot-2017.05-rc2-patch/package/x11r7/xproto_randrproto/xproto_randrproto.mk
--- buildroot-2017.05-rc2/package/x11r7/xproto_randrproto/xproto_randrproto.mk	2017-05-17 10:27:16.027844496 +0200
+++ buildroot-2017.05-rc2-patch/package/x11r7/xproto_randrproto/xproto_randrproto.mk	2017-05-27 07:30:55.000000000 +0200
@@ -10,6 +10,6 @@ XPROTO_RANDRPROTO_SITE = http://xorg.fre
 XPROTO_RANDRPROTO_LICENSE = MIT
 XPROTO_RANDRPROTO_LICENSE_FILES = COPYING
 XPROTO_RANDRPROTO_INSTALL_STAGING = YES
-XPROTO_RANDRPROTO_INSTALL_TARGET = NO
+XPROTO_RANDRPROTO_INSTALL_TARGET = YES
 
 $(eval $(autotools-package))
diff -Naurp buildroot-2017.05-rc2/package/x11r7/xproto_recordproto/xproto_recordproto.mk buildroot-2017.05-rc2-patch/package/x11r7/xproto_recordproto/xproto_recordproto.mk
--- buildroot-2017.05-rc2/package/x11r7/xproto_recordproto/xproto_recordproto.mk	2017-05-17 10:27:16.027844496 +0200
+++ buildroot-2017.05-rc2-patch/package/x11r7/xproto_recordproto/xproto_recordproto.mk	2017-05-27 07:30:55.000000000 +0200
@@ -10,6 +10,6 @@ XPROTO_RECORDPROTO_SITE = http://xorg.fr
 XPROTO_RECORDPROTO_LICENSE = MIT
 XPROTO_RECORDPROTO_LICENSE_FILES = COPYING
 XPROTO_RECORDPROTO_INSTALL_STAGING = YES
-XPROTO_RECORDPROTO_INSTALL_TARGET = NO
+XPROTO_RECORDPROTO_INSTALL_TARGET = YES
 
 $(eval $(autotools-package))
diff -Naurp buildroot-2017.05-rc2/package/x11r7/xproto_renderproto/xproto_renderproto.mk buildroot-2017.05-rc2-patch/package/x11r7/xproto_renderproto/xproto_renderproto.mk
--- buildroot-2017.05-rc2/package/x11r7/xproto_renderproto/xproto_renderproto.mk	2017-05-17 10:27:16.027844496 +0200
+++ buildroot-2017.05-rc2-patch/package/x11r7/xproto_renderproto/xproto_renderproto.mk	2017-05-27 07:30:55.000000000 +0200
@@ -10,7 +10,7 @@ XPROTO_RENDERPROTO_SITE = http://xorg.fr
 XPROTO_RENDERPROTO_LICENSE = MIT
 XPROTO_RENDERPROTO_LICENSE_FILES = COPYING
 XPROTO_RENDERPROTO_INSTALL_STAGING = YES
-XPROTO_RENDERPROTO_INSTALL_TARGET = NO
+XPROTO_RENDERPROTO_INSTALL_TARGET = YES
 
 $(eval $(autotools-package))
 $(eval $(host-autotools-package))
diff -Naurp buildroot-2017.05-rc2/package/x11r7/xproto_resourceproto/xproto_resourceproto.mk buildroot-2017.05-rc2-patch/package/x11r7/xproto_resourceproto/xproto_resourceproto.mk
--- buildroot-2017.05-rc2/package/x11r7/xproto_resourceproto/xproto_resourceproto.mk	2017-05-17 10:27:16.027844496 +0200
+++ buildroot-2017.05-rc2-patch/package/x11r7/xproto_resourceproto/xproto_resourceproto.mk	2017-05-27 07:30:55.000000000 +0200
@@ -10,6 +10,6 @@ XPROTO_RESOURCEPROTO_SITE = http://xorg.
 XPROTO_RESOURCEPROTO_LICENSE = MIT
 XPROTO_RESOURCEPROTO_LICENSE_FILES = COPYING
 XPROTO_RESOURCEPROTO_INSTALL_STAGING = YES
-XPROTO_RESOURCEPROTO_INSTALL_TARGET = NO
+XPROTO_RESOURCEPROTO_INSTALL_TARGET = YES
 
 $(eval $(autotools-package))
diff -Naurp buildroot-2017.05-rc2/package/x11r7/xproto_scrnsaverproto/xproto_scrnsaverproto.mk buildroot-2017.05-rc2-patch/package/x11r7/xproto_scrnsaverproto/xproto_scrnsaverproto.mk
--- buildroot-2017.05-rc2/package/x11r7/xproto_scrnsaverproto/xproto_scrnsaverproto.mk	2017-05-17 10:27:16.027844496 +0200
+++ buildroot-2017.05-rc2-patch/package/x11r7/xproto_scrnsaverproto/xproto_scrnsaverproto.mk	2017-05-27 07:30:55.000000000 +0200
@@ -10,6 +10,6 @@ XPROTO_SCRNSAVERPROTO_SITE = http://xorg
 XPROTO_SCRNSAVERPROTO_LICENSE = MIT
 XPROTO_SCRNSAVERPROTO_LICENSE_FILES = COPYING
 XPROTO_SCRNSAVERPROTO_INSTALL_STAGING = YES
-XPROTO_SCRNSAVERPROTO_INSTALL_TARGET = NO
+XPROTO_SCRNSAVERPROTO_INSTALL_TARGET = YES
 
 $(eval $(autotools-package))
diff -Naurp buildroot-2017.05-rc2/package/x11r7/xproto_videoproto/xproto_videoproto.mk buildroot-2017.05-rc2-patch/package/x11r7/xproto_videoproto/xproto_videoproto.mk
--- buildroot-2017.05-rc2/package/x11r7/xproto_videoproto/xproto_videoproto.mk	2017-05-17 10:27:16.027844496 +0200
+++ buildroot-2017.05-rc2-patch/package/x11r7/xproto_videoproto/xproto_videoproto.mk	2017-05-27 07:30:55.000000000 +0200
@@ -10,6 +10,6 @@ XPROTO_VIDEOPROTO_SITE = http://xorg.fre
 XPROTO_VIDEOPROTO_LICENSE = MIT
 XPROTO_VIDEOPROTO_LICENSE_FILES = COPYING
 XPROTO_VIDEOPROTO_INSTALL_STAGING = YES
-XPROTO_VIDEOPROTO_INSTALL_TARGET = NO
+XPROTO_VIDEOPROTO_INSTALL_TARGET = YES
 
 $(eval $(autotools-package))
diff -Naurp buildroot-2017.05-rc2/package/x11r7/xproto_windowswmproto/xproto_windowswmproto.mk buildroot-2017.05-rc2-patch/package/x11r7/xproto_windowswmproto/xproto_windowswmproto.mk
--- buildroot-2017.05-rc2/package/x11r7/xproto_windowswmproto/xproto_windowswmproto.mk	2017-05-17 10:27:16.027844496 +0200
+++ buildroot-2017.05-rc2-patch/package/x11r7/xproto_windowswmproto/xproto_windowswmproto.mk	2017-05-27 07:30:55.000000000 +0200
@@ -10,6 +10,6 @@ XPROTO_WINDOWSWMPROTO_SITE = http://xorg
 XPROTO_WINDOWSWMPROTO_LICENSE = MIT
 XPROTO_WINDOWSWMPROTO_LICENSE_FILES = COPYING
 XPROTO_WINDOWSWMPROTO_INSTALL_STAGING = YES
-XPROTO_WINDOWSWMPROTO_INSTALL_TARGET = NO
+XPROTO_WINDOWSWMPROTO_INSTALL_TARGET = YES
 
 $(eval $(autotools-package))
diff -Naurp buildroot-2017.05-rc2/package/x11r7/xproto_xf86bigfontproto/xproto_xf86bigfontproto.mk buildroot-2017.05-rc2-patch/package/x11r7/xproto_xf86bigfontproto/xproto_xf86bigfontproto.mk
--- buildroot-2017.05-rc2/package/x11r7/xproto_xf86bigfontproto/xproto_xf86bigfontproto.mk	2017-05-17 10:27:16.027844496 +0200
+++ buildroot-2017.05-rc2-patch/package/x11r7/xproto_xf86bigfontproto/xproto_xf86bigfontproto.mk	2017-05-27 07:30:55.000000000 +0200
@@ -10,7 +10,7 @@ XPROTO_XF86BIGFONTPROTO_SITE = http://xo
 XPROTO_XF86BIGFONTPROTO_LICENSE = MIT
 XPROTO_XF86BIGFONTPROTO_LICENSE_FILES = COPYING
 XPROTO_XF86BIGFONTPROTO_INSTALL_STAGING = YES
-XPROTO_XF86BIGFONTPROTO_INSTALL_TARGET = NO
+XPROTO_XF86BIGFONTPROTO_INSTALL_TARGET = YES
 
 $(eval $(autotools-package))
 $(eval $(host-autotools-package))
diff -Naurp buildroot-2017.05-rc2/package/x11r7/xproto_xf86dgaproto/xproto_xf86dgaproto.mk buildroot-2017.05-rc2-patch/package/x11r7/xproto_xf86dgaproto/xproto_xf86dgaproto.mk
--- buildroot-2017.05-rc2/package/x11r7/xproto_xf86dgaproto/xproto_xf86dgaproto.mk	2017-05-17 10:27:16.027844496 +0200
+++ buildroot-2017.05-rc2-patch/package/x11r7/xproto_xf86dgaproto/xproto_xf86dgaproto.mk	2017-05-27 07:30:55.000000000 +0200
@@ -10,6 +10,6 @@ XPROTO_XF86DGAPROTO_SITE = http://xorg.f
 XPROTO_XF86DGAPROTO_LICENSE = MIT
 XPROTO_XF86DGAPROTO_LICENSE_FILES = COPYING
 XPROTO_XF86DGAPROTO_INSTALL_STAGING = YES
-XPROTO_XF86DGAPROTO_INSTALL_TARGET = NO
+XPROTO_XF86DGAPROTO_INSTALL_TARGET = YES
 
 $(eval $(autotools-package))
diff -Naurp buildroot-2017.05-rc2/package/x11r7/xproto_xf86driproto/xproto_xf86driproto.mk buildroot-2017.05-rc2-patch/package/x11r7/xproto_xf86driproto/xproto_xf86driproto.mk
--- buildroot-2017.05-rc2/package/x11r7/xproto_xf86driproto/xproto_xf86driproto.mk	2017-05-17 10:27:16.027844496 +0200
+++ buildroot-2017.05-rc2-patch/package/x11r7/xproto_xf86driproto/xproto_xf86driproto.mk	2017-05-27 07:30:55.000000000 +0200
@@ -10,6 +10,6 @@ XPROTO_XF86DRIPROTO_SITE = http://xorg.f
 XPROTO_XF86DRIPROTO_LICENSE = MIT
 XPROTO_XF86DRIPROTO_LICENSE_FILES = COPYING
 XPROTO_XF86DRIPROTO_INSTALL_STAGING = YES
-XPROTO_XF86DRIPROTO_INSTALL_TARGET = NO
+XPROTO_XF86DRIPROTO_INSTALL_TARGET = YES
 
 $(eval $(autotools-package))
diff -Naurp buildroot-2017.05-rc2/package/x11r7/xproto_xf86vidmodeproto/xproto_xf86vidmodeproto.mk buildroot-2017.05-rc2-patch/package/x11r7/xproto_xf86vidmodeproto/xproto_xf86vidmodeproto.mk
--- buildroot-2017.05-rc2/package/x11r7/xproto_xf86vidmodeproto/xproto_xf86vidmodeproto.mk	2017-05-17 10:27:16.027844496 +0200
+++ buildroot-2017.05-rc2-patch/package/x11r7/xproto_xf86vidmodeproto/xproto_xf86vidmodeproto.mk	2017-05-27 07:30:55.000000000 +0200
@@ -10,6 +10,6 @@ XPROTO_XF86VIDMODEPROTO_SITE = http://xo
 XPROTO_XF86VIDMODEPROTO_LICENSE = MIT
 XPROTO_XF86VIDMODEPROTO_LICENSE_FILES = COPYING
 XPROTO_XF86VIDMODEPROTO_INSTALL_STAGING = YES
-XPROTO_XF86VIDMODEPROTO_INSTALL_TARGET = NO
+XPROTO_XF86VIDMODEPROTO_INSTALL_TARGET = YES
 
 $(eval $(autotools-package))
diff -Naurp buildroot-2017.05-rc2/package/x11r7/xproto_xineramaproto/xproto_xineramaproto.mk buildroot-2017.05-rc2-patch/package/x11r7/xproto_xineramaproto/xproto_xineramaproto.mk
--- buildroot-2017.05-rc2/package/x11r7/xproto_xineramaproto/xproto_xineramaproto.mk	2017-05-17 10:27:16.027844496 +0200
+++ buildroot-2017.05-rc2-patch/package/x11r7/xproto_xineramaproto/xproto_xineramaproto.mk	2017-05-27 07:30:55.000000000 +0200
@@ -10,6 +10,6 @@ XPROTO_XINERAMAPROTO_SITE = http://xorg.
 XPROTO_XINERAMAPROTO_LICENSE = MIT
 XPROTO_XINERAMAPROTO_LICENSE_FILES = COPYING
 XPROTO_XINERAMAPROTO_INSTALL_STAGING = YES
-XPROTO_XINERAMAPROTO_INSTALL_TARGET = NO
+XPROTO_XINERAMAPROTO_INSTALL_TARGET = YES
 
 $(eval $(autotools-package))
diff -Naurp buildroot-2017.05-rc2/package/x11r7/xproto_xproto/xproto_xproto.mk buildroot-2017.05-rc2-patch/package/x11r7/xproto_xproto/xproto_xproto.mk
--- buildroot-2017.05-rc2/package/x11r7/xproto_xproto/xproto_xproto.mk	2017-05-17 10:27:16.027844496 +0200
+++ buildroot-2017.05-rc2-patch/package/x11r7/xproto_xproto/xproto_xproto.mk	2017-05-27 07:30:55.000000000 +0200
@@ -10,7 +10,7 @@ XPROTO_XPROTO_SITE = http://xorg.freedes
 XPROTO_XPROTO_LICENSE = MIT
 XPROTO_XPROTO_LICENSE_FILES = COPYING
 XPROTO_XPROTO_INSTALL_STAGING = YES
-XPROTO_XPROTO_INSTALL_TARGET = NO
+XPROTO_XPROTO_INSTALL_TARGET = YES
 
 $(eval $(autotools-package))
 $(eval $(host-autotools-package))
diff -Naurp buildroot-2017.05-rc2/package/x11r7/xproto_xproxymanagementprotocol/xproto_xproxymanagementprotocol.mk buildroot-2017.05-rc2-patch/package/x11r7/xproto_xproxymanagementprotocol/xproto_xproxymanagementprotocol.mk
--- buildroot-2017.05-rc2/package/x11r7/xproto_xproxymanagementprotocol/xproto_xproxymanagementprotocol.mk	2017-05-17 10:27:16.027844496 +0200
+++ buildroot-2017.05-rc2-patch/package/x11r7/xproto_xproxymanagementprotocol/xproto_xproxymanagementprotocol.mk	2017-05-27 07:30:55.000000000 +0200
@@ -10,6 +10,6 @@ XPROTO_XPROXYMANAGEMENTPROTOCOL_SITE = h
 XPROTO_XPROXYMANAGEMENTPROTOCOL_LICENSE = MIT
 XPROTO_XPROXYMANAGEMENTPROTOCOL_LICENSE_FILES = COPYING
 XPROTO_XPROXYMANAGEMENTPROTOCOL_INSTALL_STAGING = YES
-XPROTO_XPROXYMANAGEMENTPROTOCOL_INSTALL_TARGET = NO
+XPROTO_XPROXYMANAGEMENTPROTOCOL_INSTALL_TARGET = YES
 
 $(eval $(autotools-package))
diff -Naurp buildroot-2017.05-rc2/package/x11r7/xserver_xorg-server/Config.in buildroot-2017.05-rc2-patch/package/x11r7/xserver_xorg-server/Config.in
--- buildroot-2017.05-rc2/package/x11r7/xserver_xorg-server/Config.in	2017-05-17 10:27:16.027844496 +0200
+++ buildroot-2017.05-rc2-patch/package/x11r7/xserver_xorg-server/Config.in	2017-05-27 08:16:07.000000000 +0200
@@ -9,6 +9,7 @@ config BR2_PACKAGE_XSERVER_XORG_SERVER
 	# We need a SHA1 implementation. If either openssl or
 	# libgcrypt are already part of the build, we'll use one of
 	# them, otherwise, use the small libsha1 library.
+	select BR2_PACKAGE_LIBDRM
 	select BR2_PACKAGE_LIBSHA1 if (!BR2_PACKAGE_OPENSSL && !BR2_PACKAGE_LIBGCRYPT)
 	select BR2_PACKAGE_MCOOKIE
 	select BR2_PACKAGE_PIXMAN
diff -Naurp buildroot-2017.05-rc2/package/x11r7/xserver_xorg-server/xserver_xorg-server.mk buildroot-2017.05-rc2-patch/package/x11r7/xserver_xorg-server/xserver_xorg-server.mk
--- buildroot-2017.05-rc2/package/x11r7/xserver_xorg-server/xserver_xorg-server.mk	2017-05-17 10:27:16.027844496 +0200
+++ buildroot-2017.05-rc2-patch/package/x11r7/xserver_xorg-server/xserver_xorg-server.mk	2017-05-27 08:16:38.000000000 +0200
@@ -13,6 +13,7 @@ XSERVER_XORG_SERVER_INSTALL_STAGING = YE
 # xfont_font-util is needed only for autoreconf
 XSERVER_XORG_SERVER_AUTORECONF = YES
 XSERVER_XORG_SERVER_DEPENDENCIES = \
+	libdrm \
 	xfont_font-util \
 	xutil_util-macros \
 	xlib_libX11 \
diff -Naurp buildroot-2017.05-rc2/package/xutil_util-macros/xutil_util-macros.mk buildroot-2017.05-rc2-patch/package/xutil_util-macros/xutil_util-macros.mk
--- buildroot-2017.05-rc2/package/xutil_util-macros/xutil_util-macros.mk	2017-05-17 10:27:16.027844496 +0200
+++ buildroot-2017.05-rc2-patch/package/xutil_util-macros/xutil_util-macros.mk	2017-05-27 07:40:31.000000000 +0200
@@ -11,7 +11,7 @@ XUTIL_UTIL_MACROS_LICENSE = MIT
 XUTIL_UTIL_MACROS_LICENSE_FILES = COPYING
 
 XUTIL_UTIL_MACROS_INSTALL_STAGING = YES
-XUTIL_UTIL_MACROS_INSTALL_TARGET = NO
+XUTIL_UTIL_MACROS_INSTALL_TARGET = YES
 
 $(eval $(autotools-package))
 $(eval $(host-autotools-package))
