diff -Naurp os-prober-1.75/linux-boot-probes/mounted/common/40grub2 os-prober-1.75-grub-initrd/linux-boot-probes/mounted/common/40grub2
--- os-prober-1.75/linux-boot-probes/mounted/common/40grub2	2017-11-25 17:20:10.117842292 +0100
+++ os-prober-1.75-grub-initrd/linux-boot-probes/mounted/common/40grub2	2017-05-20 15:23:16.391227533 +0200
@@ -78,10 +78,10 @@ parse_grub_menu () {
 				fi
 			;;
 			initrd)
-				initrd="$(echo "$2" | sed 's/(.*)//')"
-				# Initrd same.
 				if [ "$partition" != "$bootpart" ]; then
-					initrd="/boot$initrd"
+					initrd="$(echo "$line" | sed 's/initrd//;s/\t//;s|/|/boot/|g;s/(.*)//')"
+				else
+					initrd="$(echo "$line" | sed 's/initrd//;s/\t//;s/(.*)//')"
 				fi
 			;;
 			"}")
diff -Naurp os-prober-1.75/linux-boot-probes/mounted/x86/40grub os-prober-1.75-grub-initrd/linux-boot-probes/mounted/x86/40grub
--- os-prober-1.75/linux-boot-probes/mounted/x86/40grub	2017-11-25 17:20:10.120842292 +0100
+++ os-prober-1.75-grub-initrd/linux-boot-probes/mounted/x86/40grub	2017-05-20 15:21:29.154566599 +0200
@@ -66,10 +66,10 @@ parse_grub_menu () {
 			initrd)
 				# Hack alert take 2: sed off any (hdn,n)
 				# See #566102
-				initrd="$(echo "$2" | sed 's/(.*)//')"
-				# Initrd same.
 				if [ "$partition" != "$bootpart" ]; then
-					initrd="/boot$initrd"
+					initrd="$(echo "$line" | sed 's/initrd//;s/\t//;s|/|/boot/|g;s/(.*)//')"
+				else
+					initrd="$(echo "$line" | sed 's/initrd//;s/\t//;s/(.*)//')"
 				fi
 			;;
 			boot)
